<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta name="generator" content="http://www.nongnu.org/elyxer/">
<meta name="create-date" content="2015-09-06">
<link rel="stylesheet" href="http://elyxer.nongnu.org/lyx.css" type="text/css" media="all">
<title>Converted document</title>
</head>
<body>
<div id="globalWrapper">
<div class="Standard">
<div class="center">
<span class="largest"><span class="phantom"></span><span class="phantom"></span><span class="phantom"></span><span class="phantom"></span>Influence of Geographic Isolation on the Resistance of Norway spruce (<i>Picea abies</i>) to <i>Heterobasidion parviporum</i><br>
<div class="vfill"> </div></span>
</div>

</div>
<div class="Standard">
<span class="small">........................Thesis submitted for a M.Sc. degree in Forest Sciences and Business</span>
</div>
<div class="Standard">
<div class="left">
<span class="small"><span class="hspace" style="width: 66bp;"></span>University of Helsinki <br>
<span class="hspace" style="width: 66bp;"></span>Dept. of Forest Sciences <br>
<span class="hspace" style="width: 66bp;"></span>May, 2015 <br>
<span class="hspace" style="width: 66bp;"></span><br>
<span class="hspace" style="width: 66bp;"></span>Erik Stewart</span>
</div>

</div>
<div class="Standard">

</div>
<blockquote class="Quotation">
<p><br>
</p>

</blockquote>
<blockquote class="Quotation">
\begin_inset External	template PDFPages	filename C:/Users/user/Documents/Courses/Thesis/Thesis_abstract_page_Stewart.pdf\end_inset
</blockquote>
<div class="abstract">
<p class="abstract-message">
Abstract
</p>
<p><br>
</p>

</div>
<div class="abstract">
<span class="unknown">\pagenumbering</span>roman 
</div>
<div class="abstract">
<span class="unknown">\singlespacing</span>
</div>
<div class="abstract">
<div class="fulltoc">
<div class="tocheader">
Table of Contents
</div>
<div class="toc">
<a class="Link" href="#Nomenclature">Nomenclature</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Section-1">Section 1: Introduction </a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection-1.1">Subsection 1.1: Biology and Epidemiology of <i>H. annosum s.l.</i></a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.1.1">Subsubsection 1.1.1: Distribution of <i>H. annosum s.l.</i> in Europe</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.1.2">Subsubsection 1.1.2: Spread and Infection of <i>H. annosum s.l.</i></a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.1.3">Subsubsection 1.1.3: Control of <i>H. annosum s.l.</i></a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-1.2">Subsection 1.2: Defense Systems and Resistance in Conifers </a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.2.1">Subsubsection 1.2.1: Constitutive Defenses in Conifers </a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.2.2">Subsubsection 1.2.2: Inducible Defenses in Conifers</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-1.3">Subsection 1.3: <i>P. abies</i> Resistance to <i>H. annosum s.l.</i> </a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.3.1">Subsubsection 1.3.1: Host Pathogen Coevolution </a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsubsection-1.3.2">Subsubsection 1.3.2: Implications for Introduced Pathogens </a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-1.4">Subsection 1.4: Lesions as an Indication of Resistance </a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-1.5">Subsection 1.5: Mixed Effects Models in Ecology </a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-1.6">Subsection 1.6: Summary of Introduction</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Section-2">Section 2: Objectives of the Study</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection-2.1">Subsection 2.1: Central Questions</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-2.2">Subsection 2.2: Hypotheses</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Section-3">Section 3: Materials and Methods</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection-3.1">Subsection 3.1: Field Sites </a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-3.2">Subsection 3.2: Inoculum Preparation </a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-3.3">Subsection 3.3: Preparation of Study Trees and Inoculation</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-3.4">Subsection 3.4: Harvest and Processing</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-3.5">Subsection 3.5: Measurements of Lesions</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-3.6">Subsection 3.6: Analysis of Data</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Section-4">Section 4: Results</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection-4.1">Subsection 4.1: Summary Statistics for Lesions</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-4.2">Subsection 4.2: Linear Mixed Model for Lesion Geometric Response</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsubsection-4.2.1">Subsubsection 4.2.1: Model Structure</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsubsection-4.2.2">Subsubsection 4.2.2: Fixed Effects</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsubsection-4.2.3">Subsubsection 4.2.3: Random Effects</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-4.3">Subsection 4.3: Model Validation and Diagnostics</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Section-5">Section 5: Discussion</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection-5.1">Subsection 5.1: Experimental Design Limitations</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-5.2">Subsection 5.2: Potential Issues in the Analysis</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Subsection-5.3">Subsection 5.3: Uncertainty in Results</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Section-6">Section 6: Conclusions</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Appendix--1">Appendix: Appendix 1</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection--1">Subsection: Maps</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Appendix--2">Appendix: Appendix 2 </a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection--2">Subsection: Normality of Data and Outliers</a>
</div>
</div>
<div class="toc">
<a class="Link" href="#toc-Appendix--3">Appendix: Appendix 2</a>
</div>
<div class="toc">
<a class="Link" href="#toc-Appendix--4">Appendix: Appendix 3</a>
</div>
<div class="tocindent">
<div class="toc">
<a class="Link" href="#toc-Subsection--3">Subsection: Model Comparison</a>
</div>
</div>
</div>

</div>

</div>
<div class="abstract">
<a class="toc" name="Nomenclature"></a><h1 class="nomenclature">Nomenclature</h1>
</div>
<div class="abstract">
<p><br>
</p>

</div>
<pre class="LyX-Code">
<span class="unknown">\pagenumbering</span>arabic 
</pre>
<h1 class="Section">
<a class="toc" name="toc-Section-1">1</a> Introduction 
</h1>
<div class="Standard">
Norway spruce (<span class="Emph">Picea abies</span>) is a commercially significant conifer with a ubiquitous distribution throughout most of Finland. In Finland, Norway spruce accounts for 30% of total forest growing stock volume <span class="bibcites">[<a class="bibliocite" name="cite-77" href="#biblio-77"><span class="bib-index">77</span></a>]</span>. Because of its significant impact on the Finnish economy, it is important to understand and to address issues which negatively affect the growth and production of Norway spruce.
</div>
<div class="Standard">
The most devastating pathogen affecting the growth of Norway spruce in the northern hemisphere is butt rot caused by the <span class="Emph">Heterobasidion annosum</span> <i>sensu lato (s.l.)</i> species complex. Damages to forest production in Europe from <i>H. annosum s.l.</i> is greater than 800 million euros annually; as such, it is important to study the pathogen and host to better understand the complex nature of infection biology and resistance factors, which hold the potential for mitigating losses and improving forest yields in areas where <i>H. annosum s.l.</i> is prevalent <span class="bibcites">[<a class="bibliocite" name="cite-76" href="#biblio-76"><span class="bib-index">76</span></a>]</span>.
</div>
<div class="Standard">
Norway spruce established itself in northern Europe via eastern Finland approximately 6,500 years ago <span class="bibcites">[<a class="bibliocite" name="cite-60" href="#biblio-60"><span class="bib-index">60</span></a>]</span>. However, in this relatively short period of time, Norway spruce has become one of the most dominant forest trees in northern Europe, owing partially to the fact that it has high genetic plasticity <span class="bibcites">[<a class="bibliocite" name="cite-9" href="#biblio-9"><span class="bib-index">9</span></a>, <a class="bibliocite" name="cite-56" href="#biblio-56"><span class="bib-index">56</span></a>]</span>. Although most populations of <i>P. abies </i>in northern Europe have been established relatively recently considering the species emerged millions of years ago, some populations in ice free areas of western Scandinavia likely survived the last ice age approximately 18,500 years ago <span class="bibcites">[<a class="bibliocite" name="cite-69" href="#biblio-69"><span class="bib-index">69</span></a>]</span>.
</div>
<div class="Standard">
Norway spruce is a shade tolerant conifer, which grows at the final stages of ecological succession. A tolerance for shade allows <i>P. abies </i>to establish in mixed forests in the understory during early stages of growth, with increased growth opportunistically to fill in canopy gaps as they occur <span class="bibcites">[<a class="bibliocite" name="cite-29" href="#biblio-29"><span class="bib-index">29</span></a>]</span>. Initial growth of the tree is slow, but increases between 20-60 years of age <span class="bibcites">[<a class="bibliocite" name="cite-35" href="#biblio-35"><span class="bib-index">35</span></a>]</span>. Generally, <i>P. abies </i>lives for 200 years in the southern areas of its range, but can survive up to 400 years in the northern areas of its range <span class="bibcites">[<a class="bibliocite" name="cite-35" href="#biblio-35"><span class="bib-index">35</span></a>]</span>. The root systems of Norway spruce are superficial, making the tree susceptible to windfall <span class="bibcites">[<a class="bibliocite" name="cite-35" href="#biblio-35"><span class="bib-index">35</span></a>]</span>. In Finland, Norway spruce is found throughout most of the country. Only small areas in the very northern parts of Finland in the Arctic Circle are devoid of <i>P. abies. </i>Unsurprisingly, the near ubiquitous presence of <i>P. abies</i> throughout Finland and Scandinavia has led to it becoming one of the most important commercial forestry crops in northern Europe.
</div>
<div class="Standard">
Among other biotic factors, the most devastating pathogen affecting the growth of Norway spruce in the northern hemisphere is root and butt rot disease caused by the Heterobasidion annosum <i>sensu lato</i>. (<i>s.l.</i>) species complex. Damages to forest production in Europe from<i> H. annosum</i> <i>s.l</i>. is greater than 800 million euros annually. It is therefore important to study the pathogen and it’s host for a better understanding of the complex nature of the infection biology and resistance factors. The results from such studies will hold the potential for mitigating losses and improving forest yields in areas where H. annosum s.l. is prevalent (Stenlid 1998)
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-1.1">1.1</a> Biology and Epidemiology of <i>H. annosum s.l.</i>
</h2>
<div class="Standard">
The <i>H. annosum s.l. </i>species complex is comprised of several inter-sterile groups of species, each with differing host preferences. In Europe, three distinct mating types are recognized: <i>Heterobasidion annosum sensu stricto</i>, <i>Heterobasidion parviporum</i>, and <i>Heterobasidion abietinum, </i>with host preference for pines (<i>Pinus</i> <i>spp.</i>), spruce (<i>Picea spp.</i>), and firs (<i>Abies spp</i>.), respectively <span class="bibcites">[<a class="bibliocite" name="cite-34" href="#biblio-34"><span class="bib-index">34</span></a>]</span>. In North America, two <i>H. annosum s.l.</i> intersterility group are recognized; <i>Heterobasidion occidentale, </i>and <i>Heterobasidion irregulare</i>. 
</div>
<div class="Standard">
<i>H. annosum s.l</i>. has the ability to infect a broad range of coniferous trees, as well as some angiosperms. <i>H. annosum s.l.</i> are selective necrotrophs, which causes damage by degrading lignin and cellulose in host trees. Presently, no conifer tree is resistant and there is no 100% effective control against the disease. Once infection has been established within a host, death of the host tree will occur eventually due either to the pathogen or other environmental factors, such as wind throw in hosts which have compromised structural integrity. <i>H. annosum s.l. </i>are also effective saprotrophs.<i> </i>However, the degradation of host tissues to the point of lethality may take several decades. Due to the mortal nature of infections by <i>H. annosum s.l., </i>knowledge of the life-cycle and infection biology of the pathogen is critical to understanding how best to deal with the pathogen in areas where <i>H. annosum s.l. </i>presence is a problem to forestry industries. 
</div>
<div class="Standard">
Despite a preference for specific hosts, several species within the <i>H. annosum s.l.</i> complex are able to infect other than their preferred hosts, albeit with less efficiency <span class="bibcites">[<a class="bibliocite" name="cite-18" href="#biblio-18"><span class="bib-index">18</span></a>]</span>. The different species of the <i>H. annosum s.l. </i>complex<i> </i>are generally unable to hybridize with other species within the <i>H. annosum s.l </i>species complex, with genetic control over inter sterility regulated by at least five genes <span class="bibcites">[<a class="bibliocite" name="cite-18" href="#biblio-18"><span class="bib-index">18</span></a>, <a class="bibliocite" name="cite-8" href="#biblio-8"><span class="bib-index">8</span></a>]</span>. However, limited hybridization has been observed, mostly in laboratory settings between the North American isolates <span class="bibcites">[<a class="bibliocite" name="cite-16" href="#biblio-16"><span class="bib-index">16</span></a>]</span>. A study by <span class="bibcites">[<a class="bibliocite" name="cite-17" href="#biblio-17"><span class="bib-index">17</span></a>]</span> speculated that little to no observations of hybrids in natural settings is primarily due to ecological constraints, and higher competition from pure strains within the species complex. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.1.1">1.1.1</a> Distribution of <i>H. annosum s.l.</i> in Europe
</h3>
<div class="Standard">
<i>H.</i> <i>annosum s.l. </i>is widespread throughout most of the northern hemisphere. In Europe, the three intersterile groups of <i>H. annosum s.l. </i>dominate in areas where their respective host trees are found. The European P-type intersterility group (<i>H. annosum s.s.</i>) is found throughout most of Europe, with upper limits to its extent in the southern to central areas of Finland, despite the presence of suitable hosts throughout the further northern areas of Europe <span class="bibcites">[<a class="bibliocite" name="cite-33" href="#biblio-33"><span class="bib-index">33</span></a>]</span>. The S-type intersterility group (<i>H. parviporum) </i>is found further north in Europe than that of the P-type, but its southern extent generally restricted by lack of suitable hosts in the most of the more southern parts of Europe <span class="bibcites">[<a class="bibliocite" name="cite-33" href="#biblio-33"><span class="bib-index">33</span></a>]</span>. The northernmost observations of <i>H. parviporum</i> have occurred just south of the northernmost distribution of <i>P. abies</i> in Finland, at approximately 68º N. Although the S-type intersterility group has been noted in these northernmost regions, it is not considered to be a problem in mechanized forestry situations, and incidences of infections are rare. In contrast with the other two intersterility groups, the F-type intersterility species of the pathogen (<i>H. abitenium) </i>is only found in southern and central Europe, with upper limits to its distribution restricted by the availability of suitable host species<span class="bibcites">[<a class="bibliocite" name="cite-33" href="#biblio-33"><span class="bib-index">33</span></a>]</span>. However, understanding of the ecological constraints including temperature regimes which limit the distribution of these pathogens from areas where suitable hosts are found, such as in far northern Europe, is lacking <span class="bibcites">[<a class="bibliocite" name="cite-73" href="#biblio-73"><span class="bib-index">73</span></a>, <a class="bibliocite" name="cite-33" href="#biblio-33"><span class="bib-index">33</span></a>]</span>. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.1.2">1.1.2</a> Spread and Infection of <i>H. annosum s.l.</i>
</h3>
<div class="Standard">
A recent report has shown that <i>H. annosum</i> <i>s.l. </i>can stay viable for at least six years in roots of at least 15mm diameter and maintain the ability to vegetativly infect nearby roots of Norway spruce seedlings <span class="bibcites">[<a class="bibliocite" name="cite-51" href="#biblio-51"><span class="bib-index">51</span></a>]</span>. In areas where conifers are intensively harvested and managed, mechanical damages due to forestry related activities provide new surfaces for inoculations, and exacerbate the problem of <i>H. annosum s.l.</i>, leading to high rates of infection and heavy losses. The primary way in which new infections are established in areas where commercial harvesting of forest trees is done is via basidiospore deposition onto freshly cut stump surfaces <span class="bibcites">[<a class="bibliocite" name="cite-54" href="#biblio-54"><span class="bib-index">54</span></a>]</span>. The actual infection process begins when a spore or mycelia reach a suitable host substrate. Adhesion of spores requires suitable substrate, generally in the form of a fresh wound exposing living tissues in host tree, or a stump through which subsequent spread to adjacent trees can occur. Germination of spores occurs when spores land onto suitable host tissues, and environmental conditions are sufficient for the survival of the spores <span class="bibcites">[<a class="bibliocite" name="cite-54" href="#biblio-54"><span class="bib-index">54</span></a>, <a class="bibliocite" name="cite-55" href="#biblio-55"><span class="bib-index">55</span></a>]</span>. After infection has been established, <i>H. annosum</i> <i>s.l. </i>utilizes lignin and cellulose as primary carbon sources for growth and proliferation within host tissues. However, <i>H. annosum s.l. </i>can also utilize other sources of carbon as well <span class="bibcites">[<a class="bibliocite" name="cite-34" href="#biblio-34"><span class="bib-index">34</span></a>]</span>.
</div>
<div class="Standard">
Other methods by which <i>H. annosum s.l. </i>spreads is via mycelial growth within suitable host substrate, or in very limited distances throughout soils. Infections can also occur in the immature roots of host trees, as this is rare but vector by which the pathogen spreads <span class="bibcites">[<a class="bibliocite" name="cite-2" href="#biblio-2"><span class="bib-index">2</span></a>, <a class="bibliocite" name="cite-28" href="#biblio-28"><span class="bib-index">28</span></a>]</span>. <i>H. annosum s.l. </i>does not have the ability to grow far in soil without suitable host substrate. However, root systems of an infected tree which are near to neighboring tree’s root systems, or which are grafted with the roots of another tree can pass mycelium from the infected tree to the root systems of another tree. Contact between roots of neighboring trees is also an important natural vector for the transmission of <i>H. annosum s.l. </i>to uninfected trees within the same stand, and to subsequent generations in instances where stumps and remaining root tissues from infected trees are left in place <span class="bibcites">[<a class="bibliocite" name="cite-18" href="#biblio-18"><span class="bib-index">18</span></a>, <a class="bibliocite" name="cite-3" href="#biblio-3"><span class="bib-index">3</span></a>, <a class="bibliocite" name="cite-75" href="#biblio-75"><span class="bib-index">75</span></a>]</span>. 
</div>
<div class="Standard">
Reproduction of <i>H. annosum s.l. </i>requires that two homokaryotic strains of differing genetic origin and with compatible mating loci fuse to form a heterokaryotic mycelium containing the nuclei of both parental strains. With sufficient climate and access to nutrients, the heterokaryotic mycelium can form a basidiocarp, generally on the lower portion of infected trees, although several methods exist for inducing fruiting in the laboratory as well  <span class="bibcites">[<a class="bibliocite" name="cite-75" href="#biblio-75"><span class="bib-index">75</span></a>]</span>. In addition to infection from sexually produced basidiospores, conidiospores, an asexual spore produced by <i>H. annosum s.l. </i>can also start new infections, but are speculated not to be the the primary source of new infections <span class="bibcites">[<a class="bibliocite" name="cite-54" href="#biblio-54"><span class="bib-index">54</span></a>]</span>. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.1.3">1.1.3</a> Control of <i>H. annosum s.l.</i>
</h3>
<div class="Standard">
Various biotic and abiotic methods of control are used to attempt to reduce the prevalence and damage done to trees by <i>H. annosum s.l. </i>In modern, highly mechanized forest product production, one of the key routes for new infections of Heterobasidion is a byproduct of the harvest and maintenance of trees; mechanically created wounds on trees provides access to suitable host tissues for basidiospores, although most new infections occur on stumps leftover from logging operations<span class="bibcites">[<a class="bibliocite" name="cite-43" href="#biblio-43"><span class="bib-index">43</span></a>, <a class="bibliocite" name="cite-68" href="#biblio-68"><span class="bib-index">68</span></a>, <a class="bibliocite" name="cite-75" href="#biblio-75"><span class="bib-index">75</span></a>]</span>. Primary methods for control of <i>H. annosum s.l. </i>include the careful use of equipment and attempts to minimize damage due to anthropogenic factors. Post-harvest stumps left in place provide an ideal suitable host substrate for <i>H. annosum s.l. </i>Thus, the most efficient and widely utilized control methods address the issue of suitable host tissue on freshly cut stumps by means of chemical, biotic, or abiotic control in areas where infections have not been previously noted. However it is very difficult to control spread of <i>H. annosum s.l. </i>in areas where the pathogen has previously been established; in areas where prior generations were infected, stumping after the harvest of the current generation may be insufficient for controlling spread of <i>H. annosum s.l. </i>to the subsequent generations <span class="bibcites">[<a class="bibliocite" name="cite-52" href="#biblio-52"><span class="bib-index">52</span></a>, <a class="bibliocite" name="cite-53" href="#biblio-53"><span class="bib-index">53</span></a>]</span>. Chemical control includes treatment of freshly cut stumps with urea, borax, or a fungicidal product like propiconizole, which can all be effective at reducing the likelihood of subsequent infection from <i>H. annosum s.l. </i>spores <span class="bibcites">[<a class="bibliocite" name="cite-18" href="#biblio-18"><span class="bib-index">18</span></a>, <a class="bibliocite" name="cite-47" href="#biblio-47"><span class="bib-index">47</span></a>, <a class="bibliocite" name="cite-75" href="#biblio-75"><span class="bib-index">75</span></a>]</span>. A typical biotic treatments for the control of <i>H. annosum s.l. </i>includes the application of saprotrophic fungi <i>Phlebiopsis gigantea, </i>which colonizes available substrate (generally stumps) and out competes <i>Heterobasidion spp</i>. <span class="bibcites">[<a class="bibliocite" name="cite-18" href="#biblio-18"><span class="bib-index">18</span></a>, <a class="bibliocite" name="cite-47" href="#biblio-47"><span class="bib-index">47</span></a>, <a class="bibliocite" name="cite-75" href="#biblio-75"><span class="bib-index">75</span></a>]</span>. Silvicultural practices which are used to reduce the damage from <i>H. annosum s.l. </i>includes removal of stumps and associated root tissues, which provide an unchallenged substrate for infections to occur on for a period of time after harvest when stump surfaces are susceptible <span class="bibcites">[<a class="bibliocite" name="cite-71" href="#biblio-71"><span class="bib-index">71</span></a>]</span>. Harvest of stumps and removal of sources of host material suitable for inoculation has proved to be effective in reducing the subsequent infections and rot <span class="bibcites">[<a class="bibliocite" name="cite-48" href="#biblio-48"><span class="bib-index">48</span></a>]</span>.
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-1.2">1.2</a> Defense Systems and Resistance in Conifers 
</h2>
<div class="Standard">
Conifers have a variety of defense systems for dealing with both biotic and abiotic threats. The major defense systems which conifers possess for dealing with pathogens are constitutive and inducible defenses. Structural, or constitutive defenses include tissues, structures, and chemicals present in parts of the tree which either prevent or reduce the possibility or severity of infection by a pathogen. Induced defenses are initiated locally and/or systemically upon the recognition of a pathogen, and include a variety of possible defense actions including secondary metabolite production, priming of system wide defenses, and changes to physical structures of the tree. Genetic defenses represent gene level interactions between plant and pathogen, and can confer increased resistance to a pathogen or outright immunity, depending on the tree and pathogen. The constitutive and induced systems of defense present in conifers are not mutually exclusive of one another, and interact with components of the other systems of defense within the tree on some level. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.2.1">1.2.1</a> Constitutive Defenses in Conifers 
</h3>
<div class="Standard">
Constitutive defenses present throughout conifers include various types of tissues and structures which are produced during regular, unchallenged growth. Bark represents the outermost layer of defense in a conifer, and is comprised of several distinct tissues. In trees other than saplings, this is comprised of periderm, and phloem tissues. The periderm is highly suberized and hydrophobic, which helps to inhibit adhesion and germination of fungal spores <span class="bibcites">[<a class="bibliocite" name="cite-49" href="#biblio-49"><span class="bib-index">49</span></a>]</span>. In most cases, it is not possible for a fungal pathogen to actively infiltrate intact outer bark tissues, a notable exception being some species of fungi in the family <i>Armillaria </i>which can penetrate outer defenses <span class="bibcites">[<a class="bibliocite" name="cite-49" href="#biblio-49"><span class="bib-index">49</span></a>]</span>. However, not all structures within the bark can adequately resist fungal invasion; in a study conducted by <span class="bibcites">[<a class="bibliocite" name="cite-40" href="#biblio-40"><span class="bib-index">40</span></a>]</span>,<i> H. parviporum </i>was able to establish infections reaching the xylem in all test subjects with the rhytidome and phellem removed, with inner bark tissues still intact. 
</div>
<div class="Standard">
Features present in the phloem tissues which allow the <i>P. abies</i> to resist damages and pathogen includes polyphenolic parenchyma cells, and lignified cells. Polyphenolic parenchyma (PP) cells are present throughout the trees phloem tissues, and store phenolic compounds which are released if the cells are damaged by wounding either from mechanical forces or via pathogen growth. Phenolic compounds have various anti-microbial properties, and resistant clones of <i>P. abies</i> have been shown to have higher amounts of PP cells than susceptible clones <span class="bibcites">[<a class="bibliocite" name="cite-14" href="#biblio-14"><span class="bib-index">14</span></a>]</span>. Xylem tissues include some similar defense mechanisms to cells in phloem tissues. Lignified cells within the xylem increase the mechanical strength of a tree, and are more resistant to fungal infections. However, much higher portions of parenchyma cells exist within the xylem in conifers, predominantly in the form of xylem rays. PP cells are also present in large numbers in the xylem tissues. Further barriers to damage from pathogens or mechanical forces include large amounts of lignified and suberized tissues.
</div>
<div class="Standard">
Sapwood tissues have a variety of constituent and structural defenses, as well as the ability to induce defenses in response to a pathogen. For example, the sapwood tissues in Norway spruce which are more resistant to fungal infection have larger polyphenolic parenchyma cells, which can contribute to differences in resistance levels of the phloem in individual trees depending on the resistance of a given phenotype <span class="bibcites">[<a class="bibliocite" name="cite-44" href="#biblio-44"><span class="bib-index">44</span></a>]</span>. Trees also create barriers in response to pathogens, compartmentalizing both axially and radially in response to damages or detection of a pathogen. Increases to lignification near affected tissues, plugging of vascular tissues, and programmed cell death are utilized by the tree to attempt to exclude the pathogen from vulnerable areas within the host. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.2.2">1.2.2</a> Inducible Defenses in Conifers
</h3>
<div class="Standard">
In addition to the compartmentalization of tissues surrounding a wound or pathogen within the sapwood, a reaction zone surrounding initial wound is also created, which is characterized by increased levels of lignin in surrounding tissues, increased production of phytoalexins, free radical production from oxidative bursts, and increases in other chemicals which have antimicrobial properties <span class="bibcites">[<a class="bibliocite" name="cite-36" href="#biblio-36"><span class="bib-index">36</span></a>, <a class="bibliocite" name="cite-49" href="#biblio-49"><span class="bib-index">49</span></a>]</span> Increases in accumulation of antimicrobial compounds in the reaction area help to restrict further progression of pathogens or damages which caused the response and compartmentalization of the area in the first place. The hypersensitive response is the activation of programmed cellular death in localized areas in response to a pathogen; however, in the epidemiology of pathogens with necrotrophic lifestyles, such as <i>H. annosum s.l.</i>, the hypersensitive response is ineffective at stopping an infection. 
</div>
<div class="Standard">
Non-specific chemical based defenses in conifer periderm and sapwood tissues include oleoresin based defenses, increased lignification, as well as accumulation of antimicrobial chemicals which are produced during normal plant growth, referred to as phytoanticipins <span class="bibcites">[<a class="bibliocite" name="cite-70" href="#biblio-70"><span class="bib-index">70</span></a>]</span>. Oleoresins are present in sapwood tissues during the normal growth of many conifer species, and are produced by a variety of specialized cells depending on the genus. For example <i>Pinus spp. </i>have well developed resin duct tissues, whereas <i>Abies spp. </i>produce resin blisters; <i>Picea spp.</i> lack centralized traumatic resin ducts, and show low levels of constituent monoterperne cyclase activity <span class="bibcites">[<a class="bibliocite" name="cite-38" href="#biblio-38"><span class="bib-index">38</span></a>]</span>. However, oleoresin production can greatly increase in response to wounding or pathogenic attack in genus such as <i>Picea </i><span class="bibcites">[<a class="bibliocite" name="cite-38" href="#biblio-38"><span class="bib-index">38</span></a>]</span>. Because oleoresins are produced both in normal growth as well as in response to abiotic or biotic stresses, it is appropriate to consider oleoresins as an induced defense strategy as well as a constitutive one. Phytoalexins are molecular compounds produced in response to a pathogen, and include a broad range of low molecular weight compounds; however, true phytoalexins are not known in conifers, but increased accumulation of phytoanticipins and antimicrobial compounds in response to pathogens has been observed in conifers <span class="bibcites">[<a class="bibliocite" name="cite-7" href="#biblio-7"><span class="bib-index">7</span></a>]</span>. Lignans, stilbenes and terpenoids are phytoalexin-like compounds produced in <i>P. abies</i> and other <i>Picea spp</i>. in response to fungal challenge <span class="bibcites">[<a class="bibliocite" name="cite-49" href="#biblio-49"><span class="bib-index">49</span></a>]</span>. 
</div>
<div class="Standard">
Upon infection with <i>H. annosum s.l.</i>, broad changes occur in both the host and pathogen. Specific elicitors detected by a susceptible host can initiate changes to the physiology of the host both locally and systemically. Changes in the host include increased production of secondary metabolites in response to the pathogen. Similarly, once the pathogen begins to encounter resistance, it can produce compounds which assist in overcoming the host defense systems. A study by <span class="bibcites">[<a class="bibliocite" name="cite-65" href="#biblio-65"><span class="bib-index">65</span></a>]</span> found that the priming of resistance by prior inoculation of <i>H. parviporum </i>in Norway spruce had significant effects on reducing the subsequent necrosis and fungal growth in subsequent inoculations. Other factors can prime host tree defenses, prompting physiological changes which confer enhanced resistance to pathogens. Herbivory, volatile organic compounds, and colonization of the host with certain types of rhizobacteria or mychorrizal fungi also have the potential to prime defenses in conifers <span class="bibcites">[<a class="bibliocite" name="cite-12" href="#biblio-12"><span class="bib-index">12</span></a>]</span>. 
</div>
<div class="Standard">
In addition to low molecular weight compounds produced for defenses, conifers also utilize protein based defenses in response to pathogens. These protein defenses are known as pathogenesis-related proteins (PR-proteins), and include proteins across seventeen well defined families (PR1-PR17), as well as several other classes of less well understood hypothesized PR-proteins families (PR-18, PR-19) <span class="bibcites">[<a class="bibliocite" name="cite-36" href="#biblio-36"><span class="bib-index">36</span></a>, <a class="bibliocite" name="cite-72" href="#biblio-72"><span class="bib-index">72</span></a>]</span>. 
</div>
<div class="Standard">
A variety of signaling molecules are important in the induced defense in conifers. Methyl-jasmonate is an important signaling molecule (produced when a tree is under attack) which induces a wide variety of changes to conifer tissues, including traumatic resin duct formation, and increased production of terpenoid based resin defenses<span class="bibcites">[<a class="bibliocite" name="cite-42" href="#biblio-42"><span class="bib-index">42</span></a>, <a class="bibliocite" name="cite-27" href="#biblio-27"><span class="bib-index">27</span></a>, <a class="bibliocite" name="cite-24" href="#biblio-24"><span class="bib-index">24</span></a>, <a class="bibliocite" name="cite-25" href="#biblio-25"><span class="bib-index">25</span></a>]</span>. Salicylic acid is important for systemic acquired resistance (SAR), although the exact nature by which salicylic acid enables SAR is not fully understood. Finally, ethylene is an important molecule in the signaling pathways of conifer defenses, which is influenced by the production of Methyl-jasmonate <span class="bibcites">[<a class="bibliocite" name="cite-26" href="#biblio-26"><span class="bib-index">26</span></a>]</span>. Ethylene assists in defense in the phloem of conifers, and in creating traumatic resin ducts <span class="bibcites">[<a class="bibliocite" name="cite-27" href="#biblio-27"><span class="bib-index">27</span></a>]</span>. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-1.3">1.3</a> <i>P. abies</i> Resistance to <i>H. annosum s.l.</i> 
</h2>
<div class="Standard">
Norway spruce has differing levels of resistance to infection by <i>H. annosum s.l.,</i> depending on a variety of factors including genetics, tree age, and overall health <span class="bibcites">[<a class="bibliocite" name="cite-22" href="#biblio-22"><span class="bib-index">22</span></a>, <a class="bibliocite" name="cite-64" href="#biblio-64"><span class="bib-index">64</span></a>, <a class="bibliocite" name="cite-11" href="#biblio-11"><span class="bib-index">11</span></a>]</span>. <i>P. abies</i> is not known to have outright immunity to the pathogen, and, once established, disease progress will eventually kill off the host tree after degrading sufficient portions of essential tissues. Because there are no known <i>H. annosum s.l.</i> immune genotypes of <i>P. abies</i>, research efforts have focused on elucidating the factors which determine levels of resistance and susceptibility, as well as characterizing the overall interactions between the host and pathogen. Resistance factors typically examined in literature include biotic factors, such as changes to plant physiology in response to pathogens, abiotic factors, such as temperature, climate, and nutrient availability, and genetics. Abiotic factors can influence the ability of either the host of pathogen to properly defend, or infect, respectively; factors such as soil types and availability of nutrients within the soils, pH, and moisture levels have been shown to influence the resistance of <i>P. abies</i> to infection of <i>H. annosum s.l. <span class="bibcites">[<a class="bibliocite" name="cite-3" href="#biblio-3"><span class="bib-index">3</span></a>, <a class="bibliocite" name="cite-41" href="#biblio-41"><span class="bib-index">41</span></a>, <a class="bibliocite" name="cite-55" href="#biblio-55"><span class="bib-index">55</span></a>]</span>.</i>
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.3.1">1.3.1</a> Host Pathogen Coevolution 
</h3>
<div class="Standard">
Coevolution between plants and pathogens is driven by their interactions with one another over time. Differential pressures imposed by either host or pathogen onto the other shape the way in which the organisms interact with one another, and over time can lead to , resistance or susceptibility, and differing levels of virulence in the pathosystem. Theories of plant pathogen coevolution typically include the gene- for-gene model wherein one or more resistance genes (<i>R</i>) in a host are complimented with corresponding avirulence genes (<i>Avr</i>) in a pathogen. If an <i>Avr</i> gene is present in a pathogen, the corresponding gene product is recognized in an incompatible host with the corresponding resistance gene; the result is that the host recognizes the pathogen and is able to resist infection. If the necessary <i>R</i> gene is not present in the host, the outcome is a compatible reaction leading to infection. Many plant <i>R</i> gene products are nucleotide binding site leucine rich repeats (NBS-LRR), but are largely ineffective against necrotrophic pathogens, such as <i>H. annosum s.l.</i> <span class="bibcites">[<a class="bibliocite" name="cite-20" href="#biblio-20"><span class="bib-index">20</span></a>]</span>. Recent studies have examined the role of NBS-LRRs in <i>P. abies</i> in response to a necrotrophic pathogen, but findings have found only small differences in significantly upregulated gene products between wounding and infected trees <span class="bibcites">[<a class="bibliocite" name="cite-13" href="#biblio-13"><span class="bib-index">13</span></a>]</span>. Additional evidence exists for the importance of plant-pathogen coevolution in the positive selection of effective PR-proteins <span class="bibcites">[<a class="bibliocite" name="cite-58" href="#biblio-58"><span class="bib-index">58</span></a>]</span>. Specific products which have coevolved in the interactions between host and pathogen include toxins (either general, or host specific) and effectors on the pathogen side, and elicitors and other <i>R </i>gene products on the host side. Effectors act to suppress host defense systems, which in turn drives the host to the evolution of <i>R </i>gene products which recognize and neutralize the pathogen effectors. Over time, pathogens will evolve changes to their (now) unsuitable effectors to avoid the newly adapted <i>R </i>proteins. Lastly, the specific <i>R</i> or <i>Avr</i> genes do not necessarily interact directly with one another on a molecular level. In some cases, the product of a <i>R</i> gene acts as a &ldquo;guard&rdquo; to a target of the <i>Avr </i>product, and only initiates resistance when the target host protein (&ldquo;guardee&rdquo;) interacts with the pathogens <i>Avr</i> product. This interaction and subsequent elicitation of further defense mechanisms within the host of is called the guard hypothesis. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-1.3.2">1.3.2</a> Implications for Introduced Pathogens 
</h3>
<div class="Standard">
Overall, coevolution is an important driving factor in the development of resistant strains of a plant. However, in instances where a potential host has been excluded from the presence of pathogen, resistance of the host may be greatly reduced or nonexistent all together for the newly introduced pathogen. This interaction between susceptible host and non-native introduced pathogen can cause devastating damages to a species which has not had the chance to coevolve alongside the pathogen. Perhaps the best known case of an introduced pathogen having devastating effects on a new host is the introduction of <i>Cryphonectria parasitica, </i>a fungal pathogen from Asia, into the eastern areas of North America in the early 1900’s <span class="bibcites">[<a class="bibliocite" name="cite-1" href="#biblio-1"><span class="bib-index">1</span></a>]</span>. Due to the introduction of<i> C. parasitica</i>, the causal agent of chestnut blight, the American chestnut was reduced from a once dominant species across much of the eastern United States to a mere pittance<span class="bibcites">[<a class="bibliocite" name="cite-1" href="#biblio-1"><span class="bib-index">1</span></a>]</span>. Other pathogens which have been introduced and caused great damages to native hosts include <i>Phytophthora ramorum, </i>the agent responsible for the sudden oak death, and various <i>Ophiostoma</i> species, which have caused various epidemics of Dutch elm disease in both North America and Europe. 
</div>
<div class="Standard">
Differences in generation times between host and pathogen can also influence the coevolution of the species. For example, <i>P. abies,</i> as with most coniferous trees, has long generational periods, versus <i>H. annosum s.l., </i>which generally reproduces much quicker, leading to faster adaptations by the pathogen. The shorter life cycles of the pathogen, as well as both sexual, and asexual reproduction modes allows subsequent generations to select quickly for favorable traits and increased virulence against the host with a slower generation time <span class="bibcites">[<a class="bibliocite" name="cite-19" href="#biblio-19"><span class="bib-index">19</span></a>]</span>. However, in the interactions between <i>H. annosum s.l. </i>and <i>P. abies</i>, resistance of the host and virulence of the pathogen are quantitative traits, which are under the control of many different genes. Specific genes in <i>P. abies </i>which control for resistance to <i>H. annosum s.l. </i>are not fully known. However, recent efforts have made some progress towards identifying promising regions in the genome of <i>P. abies </i>for quantitative resistance traits <span class="bibcites">[<a class="bibliocite" name="cite-39" href="#biblio-39"><span class="bib-index">39</span></a>]</span>. Finally, it is important to understand that most coevolutionary processes happen continuously in the context of host and pathogen interactions; either host or pathogen may eventually adapt to overcome the defenses or offenses of their compliment, changing the evolutionary direction of the compliment to attempt to adapt to the new challenge.  
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-1.4">1.4</a> Lesions as an Indication of Resistance 
</h2>
<div class="Standard">
Measuring necrotic lesions produced in response to <i>H. annosum s.l. </i>is a technique to gauge potential resistance of hosts to fungal infection, and has been utilized in many studies <span class="bibcites">[<a class="bibliocite" name="cite-11" href="#biblio-11"><span class="bib-index">11</span></a>]</span>. Lesions, which are a visible part of the reaction zone, are created in response to wounding, or from infection via a pathogen by the host tree. In cases where a pathogen, i.e., <i>H. annosum s.l., </i>is placed into the tree, the size of the resulting lesion can be used as one measure to gauge the ability of the host genet to resist the pathogen <i><span class="bibcites">[<a class="bibliocite" name="cite-74" href="#biblio-74"><span class="bib-index">74</span></a>, <a class="bibliocite" name="cite-67" href="#biblio-67"><span class="bib-index">67</span></a>, <a class="bibliocite" name="cite-11" href="#biblio-11"><span class="bib-index">11</span></a>]</span>. </i>Inoculations are generally performed utilizing a sterilized wooden dowel which is then cultured with the pathogen or left sterile as a control sample. The dowels are then placed into the host in a systematic way, and left <i>in situ </i>for a period of time before measuring the resulting lesions. Most inoculation experiments with <i>H. annosum s.l.</i> have focused on stem inoculations. However, as the roots represent the natural infection pathway for the pathogen, more research into the differences between lesions response in the stems and roots could be beneficial to better understanding pathosystem dynamics. 
</div>
<div class="Standard">
Potential factors affecting resistance traits of individual trees are influenced by genetics, and resistance can be a quantitative trait or absolute depending on the pathosystem. For example, white pine blister rust caused by <i>Cronartium ribicola</i> is an invasive pathogen, and resistance developments via breeding programs for this pathosystem are largely based on quantitative traits. Conversely, a native fusiform rust caused by <i>Cronartium quercuum</i> f. sp. <i>fusiforme</i> which infects various pines in North America more often encounters total genetic resistance to infection versus the invasive <i>Cronartium ribicola</i> <span class="bibcites">[<a class="bibliocite" name="cite-63" href="#biblio-63"><span class="bib-index">63</span></a>]</span>. Furthermore, a study by <span class="bibcites">[<a class="bibliocite" name="cite-46" href="#biblio-46"><span class="bib-index">46</span></a>]</span> found that resistance in Scots pine seedlings to artificial infection with <i>H. annosum s.s. </i>was higher when seeds were sourced from naturally regenerated forests with high natural incidences of root rot, suggesting some amount of heritability of quantitative resistance factors.
</div>
<div class="Standard">
Broad sense heritability, <span class="formula"><i>H</i><sup>2</sup></span>, is defined as the variation of the genotype in question divided by the variation of the phenotype. Overall variance in broad sense heritability of the fungal extension in seventeen year old Norway spruce clones artificially inoculated with <i>H. parviporum</i> in a study by <span class="bibcites">[<a class="bibliocite" name="cite-66" href="#biblio-66"><span class="bib-index">66</span></a>]</span> was estimated to be 0.18 <span class="formula"><i>H</i><sup>2</sup></span>, indicating that genetics do play a potential role in the resistance of Norway spruce to <i>H. parviporum.</i> An earlier study by <span class="bibcites">[<a class="bibliocite" name="cite-64" href="#biblio-64"><span class="bib-index">64</span></a>]</span> similarly found fungal growth and lesion size broad sense heritability for Norway spruce clones artificially inoculated with <i>H. parviporum</i> to be 0.35\strikeout off\uuline off\uwave off <span class="formula"><i>H</i><sup>2</sup></span><span class="default">\uuline default\uwave default and 0.27 <span class="formula"><i>H</i><sup>2</sup></span>, respectively, so variability depends on multiple factors and is not consistent. Although these studies have indicated that heritability and genetics have an influence on the resistance of <i>P. abies</i> to <i>H. annosum s.l., </i>which genes in particular have the most influence on resistance are not fully known.</span>
</div>
<div class="Standard">
 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-1.5">1.5</a> Mixed Effects Models in Ecology 
</h2>
<div class="Standard">
The mixed effect model is an extension of a general linear model which incorporates both fixed and random effects. Fixed effects are treatments, or experimental conditions which are known and controlled for in the design of the experiment. Random effects are the effects of grouping or clustering present in the data. For example, in a study utilizing multiple randomized sites, it would also be appropriate to include site as a random effect. Subject units as a group can also be considered as a random effect, such as randomly selected trees within a sample plot, as well as any subsequent levels of groupings related to an individual in a group, e.g., multiple measurements by tree, or additional levels of subgrouping. A further benefit to the use of mixed effects modeling is the ability to deal with repeated measures over time. Finally, the mixed effects model addresses issues with non-normal data, or data from experiments which are unbalanced, data which are correlated, or other shortcomings in data which render it unsuitable for analysis utilizing standard statistical techniques such as ANOVA or simple linear models <span class="bibcites">[<a class="bibliocite" name="cite-6" href="#biblio-6"><span class="bib-index">6</span></a>]</span>. Mixed effects models have been used in limited amounts in studies of<i> H. annosum</i> <i>s.l. </i>and its effects upon host trees, i.e., <span class="bibcites">[<a class="bibliocite" name="cite-66" href="#biblio-66"><span class="bib-index">66</span></a>, <a class="bibliocite" name="cite-31" href="#biblio-31"><span class="bib-index">31</span></a>]</span>. In this study, the analysis of the collected data is done with a mixed effect model allowing for the interpretation of not only fixed effects on lesion size, but also a broad analysis of the different variation that exists within the study due to repeated measurements and nesting, which could not be done concisely with other techniques such as ANOVA. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-1.6">1.6</a> Summary of Introduction
</h2>
<div class="Standard">
This study addresses several aspects of <i>H. parviporum </i>infections in <i>P. abies </i>which help to address the lack of knowledge about certain elements of the epidemiology of <i>H. annosum s.l. </i>root rot in <i>P. abies</i>. First, the study examines the potential for influence of historical isolation of <i>P. abies</i> from <i>H. parviporum</i>, and seeks to address whether a lack of coevolution in recent history could influence the ability of the host to defend against the pathogen. Secondly, this study address the variation in lesions produced in both roots and the stem organs of the host tree. This study also addresses differences in variation based on the different tissues in both stem and root organs of the trees. This study utilizes non-clonal genotypes for experimental units, helping to address and understand the variations in resistance that would be expected in natural populations of <i>P. abies. </i>Lastly, the analysis for this study is done utilizing a mixed effect model which takes into account the inherent nesting and hierarchy of the data collected, and presents the variation observed due to both fixed and random effects.
</div>
<div class="Standard">
<p><br>
</p>

</div>
<h1 class="Section">
<a class="toc" name="toc-Section-2">2</a> Objectives of the Study
</h1>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-2.1">2.1</a> Central Questions
</h2>
<div class="Standard">
This study seeks to answer several questions about the potential resistance of <i>P. abies </i>to <i>H. parviporum </i>infections in natural settings. Primarily, the study will attempt to characterize as robustly as feasible the natural variation in resistance (as determined by lesion size measurements) to infection by <i>H. parviporum </i>in the roots, stem, phloem, and xylem tissues in naturally populated, non-clonal stands across two sites in Finland. Secondly, the study will attempt to determine if variation between sites with a history of the pathogen being either present or absent is a significant explanation of the difference in observed in lesion sizes and effects in the respective field sites. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-2.2">2.2</a> Hypotheses
</h2>
<div class="Standard">
<i>Hypothesis 1: Lesion sizes in the northern field site will be larger. </i>
</div>
<div class="Standard">
Differences in growing site location, as well as differences in genetic makeup of the stands in the two sites differ, and it is presumed that these differences will have an effect on the characteristics of the lesions at the site level. Because it is believed that the northern area in Rovaniemi utilized for the study has been largely isolated from the pressures of the pathogen, it is assumed that this will decrease resistance response to the pathogen in comparison to trees at the more southern Lapinjärvi site. This effect was tested by inoculations in a balanced design experiment across the two field sites, and further analyzed by utilizing a mixed effects model to characterize the lesions in a hierarchical framework accounting for inherent nesting and lack of heteroscedasticity in data collected from the experiment. 
</div>
<div class="Standard">
<i>Hypothesis 2: Lesion sizes in the roots will be larger than in stems</i>
</div>
<div class="Standard">
Due to the different anatomical features present in the respective organs, as well as taking into account the natural infection pathways of <i>H. annosum s.l., </i>it is presumed that a difference in the response will manifest as larger lesions within the roots because it is the natural infection pathway for which the pathogen is adapted. This effect is tested using the mixed effects model, and further elaborated upon by exploring potential interactions between the effect of organ and other explanatory variables. 
</div>
<div class="Standard">
<i>Hypothesis 3: Lesions sizes in xylem tissues will be lower than those in phloem.</i>
</div>
<div class="Standard">
Due to different anatomical features in xylem and phloem tissues, it is presumed the host will respond differently to the infection in different tissues. It is hypothesized that the xylem tissues will present with smaller lesions due to the fact that high amounts of lignified tissues will present a greater challenge for the pathogen to overcome during the incubation time for the experiment compared to softer living tissues within the phloem. This effect is tested with the mixed effect model, and further elaborated upon by examining interactions between the effects of tissue in conjunction with other explanatory variables. 
</div>
<div class="Standard">
<p><br>
</p>

</div>
<h1 class="Section">
<a class="toc" name="toc-Section-3">3</a> Materials and Methods
</h1>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-3.1">3.1</a> Field Sites 
</h2>
<div class="Standard">
Two sites were used in this study, one in northern Finland (Rovaniemi) where presence of <i>H. parviporum</i> was not known to occur naturally in Norway spruce stands, and a site in Southern Finland in Lapinjärvi, where extensive presence of the pathogen has been noted historically. In Rovaniemi, the site was comprised of planted trees with an age of around 35 years, from natural, non-clonal stock representative of the natural local populations of Norway spruce. In Lapinjärvi, the site was comprised of naturally regenerated trees (non-planted), with estimates of age at 20 years or less. Soils in the northern site were rockier, versus softer soils in the southern field site. Trees in Rovaniemi had visibly significantly smaller root systems than those in Lapinjärvi, although no measurements were taken. At each of the two field sites, a total of fifteen trees were selected for each treatment group; wounding control, and infection. Trees which were obviously damaged due to biotic or anthropogenic factors, or trees which were obviously extremely young, or otherwise unhealthy were excluded from selection. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-3.2">3.2</a> Inoculum Preparation 
</h2>
<div class="Standard">
The culture of <i>H. parviporum</i>, isolate #03014 (Kari Korhonen) was used for this study. The isolate was obtained from a Norway spruce tree in Kuhmoinen, central Finland. The culture was maintained on 2% malt extract agar and kept at 4º Celsius. To create a delivery vessel for the inoculum, wooden dowels of Norway spruce measuring roughly 6 mm diameter by 7 mm length were created utilizing a drill press and a jewelers saw. Cut and formed dowels were autoclaved for thirty minutes with roughly 100 milliliters of Milli-Q water. Subsequently, autoclaved dowels for control trees were placed onto previously prepared 2% malt extract agar plates, and incubated at room temperature for two weeks. Autoclaved dowels utilized for infection were placed onto 2% malt extract agar plates which were pre-colonized with the <i>H. parviporum</i> isolate, and were similarly incubated at room temperature for two weeks. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-3.3">3.3</a> Preparation of Study Trees and Inoculation
</h2>
<div class="Standard">
Inoculations in Lapinjärvi were carried out the 14th of June, 2013. In Rovaniemi, inoculations were carried out on the 26th of June, 2013. 
</div>
<div class="Standard">
For this study, each tree was treated with a total of six inoculations; three stem inoculations placed at 50 cm, 100 cm, and 150 cm above the soil level, and three in the roots placed opportunistically depending on the size and shape of the exposed roots. Whenever possible the inoculations were placed on the parts of the root facing upwards towards the crown of the tree. A minimum of 25 cm upwards or downwards was maintained whenever possible between root inoculations in instances where a single large root was inoculated more than once in an attempt to ensure separate infections would not overlap during the study. In all trees, the inoculations were performed such that the inoculation point of the stem inoculations was at the same direction to minimize any possible variance due to azimuth. 
</div>
<div class="Standard">
Prior to inoculations i.e. insertion of wounding (termed as control) dowels or <i>H. parviporum </i>colonized (termed as infection) dowels, roots of the study trees were dug up carefully using garden spades, and then outer surfaces of exposed roots were cleaned of excess dirt and debris utilizing a large brush with synthetic fiber hairs. Directly before inoculating the trees, the area where the inoculation were to be placed was sprayed with 70% ethanol to minimize the possibility of introducing localized contaminants into the xylem or phloem of the tree. All tools utilized in the inoculation process which had direct contact with either the host tree, or wooden inoculation dowels (and thus, potentially <i>H. parviporum</i>) were sprayed with 70% ethanol and wiped clean prior to each use. Tools used for inoculations included: stainless steel forceps, for handling of the wooden dowel, and removing any excess original host tissues, a 7.0 mm interior diameter stainless steel punch for creating the bore and removing host tissue, and a large rubber headed mallet, utilized for hammering in the punch. 
</div>
<div class="Standard">
For each stem inoculation, a steel punch with a diameter of 7.0 mm was used to bore through the bark of the tree to the inoculation point. Living host tissue from the cavity created by the bore was removed, and then inspected to ensure that the depth of the hole bored reached into the xylem of the host tree. Confirmation of sufficient depth was done by inspecting the tissue removed from the bore; a clear delineation exists between phloem and xylem tissues in <i>P. abies</i>. Immediately after confirmation of adequate bore depth, a wooden dowel for the specific treatment for the tree was introduced into the hole and secured into host tissue via use of forceps and the back end of the bore. Immediately after the dowel was secured into the host tissues, the area of the inoculation was wrapped with parafilm to minimize the chance of post-inoculation contamination. After trees were inoculated and inoculation points were wrapped with parafilm, the roots were covered with the loose soil removed to expose the roots for inoculations. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-3.4">3.4</a> Harvest and Processing
</h2>
<div class="Standard">
After three months, inoculated trees were cut down for processing. Roots of the respective trees were re-dug carefully and extracted. Primary tools utilized in harvest were chainsaws. Only the lower segment of the stem containing the inoculation points were removed from the site for further processing. In root samples, as much as was feasible and reasonable was collected for each inoculation. Samples were labeled with the site (L, R, for Lapinjärvi or Rovaniemi respectively), treatment (W, for wounding control, T, for treatment), tree (1-15), and replicate number, (A, B, and C for stem representing 50, 100, and 150 cm inoculation points, and R1, R2, R3 for root inoculations, with R1 being the closest inoculation point to the root collar) before storing. Additionally, the direction of growth (i.e., towards crown) was marked on stem segments. 
</div>
<div class="Standard">
All samples were placed into large black trash bags (Lapinjärvi) or Kevlar sacks (Rovaniemi) once trimmed to size at the respective field site. Samples from Lapinjärvi were taken directly to the University of Helsinki and placed into cold storage at -18°C. Samples from Rovaniemi were placed in the -40°C freezer at the METLA Rovaniemi field station, and then shipped under temperature control to the University of Helsinki. Upon arrival, the samples from Rovaniemi were placed in cold storage at -18°C until further processing.
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-3.5">3.5</a> Measurements of Lesions
</h2>
<div class="Standard">
Before lesion measurements and analysis, the samples were processed. First, bark tissue surrounding the inoculation point on any given sample was removed utilizing a hatchet, knife, scalpel, or other suitable tools. Care was taken not to damage phloem tissues underneath. In general, at least 10 cm of bark was cleared both above and below the inoculation point, with additional areas cleared if the lesion extended beyond the original margins of removed bark on the phloem. To the left or right of the inoculation point, 5 cm of bark was removed, again removing further tissue if lesions extended beyond the margins. After bark was removed, the initial wounding and lesion was photographed and measured in the exposed phloem tissue. A total of five measurements were taken for each lesion: length of lesion extending upwards (towards the crown in stem samples, or towards the root collar in root samples) from the uppermost part of the wooden inoculation dowel, length of lesion extending downward from the lowermost point of the inoculation, length of lesion extension to the left and right of the inoculation point, and a total length measurement taken from the uppermost point of lesion extension upwards to the lowermost point of lesion extension downwards (see Figure 1). After measurements of the phloem sample, phloem tissue was removed utilizing a mallet, scalpel, and woodworking chisels to expose the xylem tissue. Lesions present in the xylem tissue were photographed and measured in the same manner as the phloem tissue.<div class="float">
<a class="Label" name="Figure-1"> </a><div class="figure">
<img class="embedded" src="schematicswip.png" alt="figure schematicswip.png" style="width: 384px; max-width: 960px; height: 288px; max-height: 720px;">
<span class="footnotesize"><i><div class="caption">
Figure 1 <span class="footnotesize">Schematic representation of inoculations (A) on stem and roots, and measurements taken (B) from resulting lesions after the experiment. In (B), measurements are for lesions extension upwards from inoculation dowel (1), extension downwards from inoculation dowel (2), extension to right or left from inoculation dowel (3,4), and total length of the lesion (5) . The red circle in (B) outlines the inoculation dowel. </span>
</div>
</i></span>
</div>

</div>

</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-3.6">3.6</a> Analysis of Data
</h2>
<div class="Standard">
Preprocessing of the data was necessary to properly format the data, and to ensure that the data was suitable for subsequent statistical analysis. For the purposes of the analysis of the size of visible lesions, i.e. the total length of the lesion and total width of the lesion are used as the primary response variables. To create a single response variable with which to characterize the lesions of the individual tree, the lesions are conceptualized as 2-dimensional rectangles based on the total length and maximal width of the lesion as measured from the center of the inoculation point. 
</div>
<div class="Standard">
Let <span class="formula"><i>x</i><sub>1</sub></span> be the total length of the lesion for a given measurement and <span class="formula"><i>x</i><sub>2</sub></span> be the width. The geometric response, <span class="formula"><i>y</i></span> is described by equation <a class="Reference" href="#eq:1">1↓</a>.
</div>
<div class="Standard">
<div class="formula">
<a class="eqnumber" name="eq:1">(1) </a><i>y</i> = (<i>x</i><sub>1</sub><i>x</i><sub>2</sub>)
</div>

</div>
<div class="Standard">
Analysis of the data  was conducted using R 3.1.2 (R Core Team, 2014), and packages lme4 <span class="bibcites">[<a class="bibliocite" name="cite-5" href="#biblio-5"><span class="bib-index">5</span></a>]</span>. Summary statistics and general overview of the data is available in the results section. Welch’s two sample t-test was used to determine if differences existed between the different directions of lesion growth. A linear mixed effects model was used to determine the effects that the categorical variables of site, tree, organ, and tissue had on tree response to the treatment. As repeated measurements from a single subject (e.g., tree) cannot be assumed to be independent of one another, explicit nesting was specified in building the model to ensure accurate interpretation of the model results. Linear mixed effects models allow for the broad analysis of data which takes on complex, interrelated levels. 
</div>
<div class="Standard">
The mixed effects model takes the form:
</div>
<div class="Standard">
<span class="large"><div class="formula">
<a class="eqnumber" name="Eq 2: The Mixed Effect Model">(2) </a><i>y</i> = <i>X</i><i>β</i> + <i>Z</i><i>γ</i> + <i>ϵ</i>
</div>
</span>
</div>
<div class="Standard">
where \strikeout off\uuline off\uwave off<span class="formula"><i>y</i></span><span class="default">\uuline default\uwave default is the<i> </i>model estimate for the expected <b>Geometric<i> Response</i></b>,</span><span class="large"> <span class="formula"><i>X</i></span></span><span class="default"> is the design matrix of the observed fixed effects variables (<b><i>Site, Treatment, Organ, Tissue)</i></b>,</span><span class="large"> <span class="formula"><i>β</i></span></span><span class="default"> is the vector of the fixed effect coefficients, </span><span class="large"><span class="formula"><i>Z</i></span></span><span class="default"> is the design matrix of random effect variables (<b><i>Tree, Organ </i></b>within<b> Tree, Sample </b>within<b> Organ </b>within<b> Tree)</b>,</span><span class="large"> <span class="formula"><i>γ</i></span> </span><span class="default">is the vector of the random effects coefficients, and </span><span class="large"><span class="formula"><i>ϵ</i></span></span><span class="default"> is a vector of residual errors. The responses are assumed to be drawn from a normal distribution. </span>
</div>
<div class="Standard">
Graphics were composed with R 3.1.2 and packages ggplot2, lattice, and gridExtra <span class="bibcites">[<a class="bibliocite" name="cite-4" href="#biblio-4"><span class="bib-index">4</span></a>, <a class="bibliocite" name="cite-21" href="#biblio-21"><span class="bib-index">21</span></a>, <a class="bibliocite" name="cite-57" href="#biblio-57"><span class="bib-index">57</span></a>]</span>.. Exploratory analysis with histograms and other charting techniques was utilized to explore the structure and nature of the data along all steps of modeling. A discussion of potential outliers as well as the effects they have on the outcome of the model is given in the appendix; because it was not possible to address the causes for several outliers present in the data, the data was left intact for the primary analysis. To check normality of the response variable, \strikeout off\uuline off\uwave off<span class="formula"><i>y</i></span><span class="default">\uuline default\uwave default, a Q-Q plot was created, and when data visually violated assumptions of normality, a log natural transform was used to attempt to bring the data into conformation of the normal distribution, and a subsequent histogram was utilized to verify the changes in the response variable. In elementary statistics comparing the sizes of lesion measurements, a constant of 1.0 mm was added to all measurements to correct in rare instances where no visible lesion growth presented in one of the directions measured. <p><br>
</p>
</span>
</div>
<h1 class="Section">
<a class="toc" name="toc-Section-4">4</a> Results
</h1>
<div class="Standard">
Summary of Lesion Measurements Taken
</div>
<div class="Standard">
SITE
</div>
<div class="Standard">
Trees in the northern field site were found to be susceptible to infection from <i>H. parviporum. </i>Average lesion size expressed as geometric response for trees in the Rovaniemi field site were <span class="formula">6.0±0.7<i>mm</i><sup>2</sup></span> versus <span class="formula">6.0±0.8<i>mm</i><sup>2</sup></span> in Lapinjarvi (Table 3); no significant difference exists in mean size of the geometric response between field sites (<i>F=0.14, d.f.=1, p=0.71</i>). Mean lesion extending upwards does not vary between sites (<i>F=0.20, d.f.=(1, 718), p=0.65</i>2), nor does the mean extension downwards <i>(F=0.463, d.f.= (1,718), p= 0.49). </i>Mean width of lesions do not vary according to site (<i>F=0.02, d.f.=(1,718), p=0.88</i>). 
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Table-1"> </a><div class="table">
<div class="caption">
Table 1 Mean Geometric Lesion Response, <span class="formula"><i>y</i></span>
</div>
<div class="center">
<table>
<tr>
<td align="center" valign="top" colspan="2">
<b>Lesion Geometric Response </b>
</td>

</tr>
<tr>
<td align="center" valign="top">
<b>Site</b>
</td>
<td align="center" valign="top">
<b>Mean <span class="formula">(<i>mm</i><sup>2</sup>)</span> <span class="formula">±<i>SD</i></span></b>
</td>

</tr>
<tr>
<td align="center" valign="top">
<i>Lapinjärvi</i>
</td>
<td align="center" valign="top">
<i><span class="footnotesize"><span class="formula">6.0±0.8</span></span></i>
</td>

</tr>
<tr>
<td align="center" valign="top">
<span class="footnotesize"><i>Infected</i></span>
</td>
<td align="center" valign="top">
<i><span class="footnotesize"><span class="formula">6.4±0.6</span></span></i>
</td>

</tr>
<tr>
<td align="center" valign="top">
<span class="footnotesize"><i>Control</i></span>
</td>
<td align="center" valign="top">
<i><span class="footnotesize"><span class="formula">5.4±0.6</span></span></i>
</td>

</tr>
<tr>
<td align="center" valign="top">
<i>Rovaniemi</i>
</td>
<td align="center" valign="top">
<i><span class="footnotesize"><span class="formula">6.0±0.7</span></span></i>
</td>

</tr>
<tr>
<td align="center" valign="top">
<span class="footnotesize"><i>Infected</i></span>
</td>
<td align="center" valign="top">
<i><span class="footnotesize"><span class="formula">6.4±0.7</span></span></i>
</td>

</tr>
<tr>
<td align="center" valign="top">
<span class="footnotesize"><i>Control</i></span>
</td>
<td align="center" valign="top">
<i><span class="footnotesize"><span class="formula">5.5±0.5</span></span></i>
</td>

</tr>

</table>

</div>

</div>

</div>

</div>
<div class="Standard">
Lesion size and response within tree organs: 
</div>
<div class="Standard">
Mean lesion sizes of the geometric response of the roots and stems were <span class="formula">6.0±0.7<i>mm</i><sup>2</sup></span> and <span class="formula">5.9±0.8<i>mm</i><sup>2</sup></span> respectively, and did not differ significantly between organs (<i>F=0.49, d.f.=1, p = 0.47</i>). In Rovaniemi, the mean size of the lesions as measured by geometric response in the stems is <span class="formula">6.0±0.8<i>mm</i><sup>2</sup></span> and in the roots is <span class="formula">5.9±0.7<i>mm</i><sup>2</sup></span>. In Lapinjärvi, mean sizes for roots and stems were <span class="formula">6.0±0.7<i>mm</i><sup>2</sup></span>and <span class="formula">6.8±0.9<i>mm</i><sup>2</sup></span>, respectively (Table 1). Mean lesion size within organ does vary by site, with Rovaniemi stems expressing larger lesions than those in Lapinjarvi (<i>F=5.99</i>, <i>d.f. =1, p=</i> <i>0.02</i>). 
</div>
<div class="Standard">
TISSUE
</div>
<div class="Standard">
Mean lesion sizes for tissues across sites were 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-4.1">4.1</a> Summary Statistics for Lesions
</h2>
<div class="Standard">
The mean lesion sizes are presented in Table 1. The mean lesions upwards are generally consistent across sites. However, downwards lesions are slightly larger for infected samples in Rovaniemi versus Lapinjärvi, although the difference is not significant (<i>t</i>=-0.40, <i>D.F</i>. =717.334, <i>p</i>=0.68). Width is higher in Lapinjärvi infected trees, but the difference is not significant (<i>t=</i>-0.96, <i>D.F</i>. =693.57,<i> p=</i>0.33). There is no difference between the mean sizes of the upwards or downwards lesion measurements (<i>t=</i>1.31<i>, D.F</i>. =1432.67, <i>p=0.18). </i>
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Table-2"> </a><div class="table">
<div class="caption">
Table 2 Mean Lesion Measurements <span class="formula">±</span>Standard Deviation
</div>
<span class="scriptsize"><table>
<tr>
<td align="right" valign="top" colspan="6">
<b>Lesions Extensions in Phloem</b>
</td>

</tr>
<tr>
<td align="right" valign="top">
<b><span class="scriptsize">Site</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Upwards (mm)</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Downwards (mm)</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Width (mm)*</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Total Length (mm)*</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Geom. Resp. mm<span class="formula"><sup>2</sup></span>)*</span></b>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Lapinjärvi</i></span>
</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Infected</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">19.1±14.9</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">17.6±8.9</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">16.0±5.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">45.5±21.8</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">6.70±0.56</span></span>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Control</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">6.8±7.4</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">8.4±12.7</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">12.0±4.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">22.2±17.1</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">5.70±0.59</span></span>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Rovaniemi</i></span>
</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Infected</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">17.6±13.1</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">18.4±10.6</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">12.0±5.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">48.1±25.9</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">6.58±0.68</span></span>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Control</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">5.7±4.5</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">6.0±5.7</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">12.0±3.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">20.4±8.6</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">5.62±0.50</span></span>
</td>

</tr>

</table>
</span><div class="PlainVisible">
<span class="small"><i>*Measurement includes the inoculation dowel width as well as lesion extension.</i></span>
</div>

</div>

</div>

</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Table-3"> </a><div class="table">
<div class="caption">
Table 3 Mean Lesion Measurements <span class="formula">±</span>Standard Deviation
</div>
<span class="scriptsize"><table>
<tr>
<td align="right" valign="top" colspan="6">
<b>Lesion Extensions in Xylem</b>
</td>

</tr>
<tr>
<td align="right" valign="top">
<b><span class="scriptsize">Site</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Upwards (mm)</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Downwards (mm)</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Width (mm)*</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Total Length (mm)*</span></b>
</td>
<td align="right" valign="top">
<b><span class="scriptsize">Geom. Resp. (mm<span class="formula"><sup>2</sup></span>)</span></b>*
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Lapinjärvi</i></span>
</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Infected</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">19.6±15.0</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">16.0±9.3</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">16.0±5.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">45.5±21.8</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">6.19±0.55</span></span>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Control</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">5.0±5.8</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">6.4±11.3</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">12.0±4.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">22.2±17.1</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">5.14±0.59</span></span>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Rovaniemi</i></span>
</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>
<td align="right" valign="top">

</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Infected</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">20.4±16.2</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">20.0±14.8</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">12.0±5.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">48.1±25.9</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">6.30±0.63</span></span>
</td>

</tr>
<tr>
<td align="right" valign="top">
<span class="footnotesize"><i>Control</i></span>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">5.0±3.8</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">6.4±6.6</span></span></i>
</td>
<td align="right" valign="top">
<i><span class="footnotesize"><span class="formula">12.0±3.0</span></span></i>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">20.4±8.6</span></span>
</td>
<td align="right" valign="top">
<span class="footnotesize"><span class="formula">5.34±0.42</span></span>
</td>

</tr>

</table>
</span><div class="PlainVisible">
<span class="small"><i>*Measurement includes the inoculation dowel width as well as lesion extension.</i></span>
</div>

</div>

</div>

</div>
<div class="Standard">
Mean lesion size of the geometric response variable overall for site, as well as for treatments within the site are presented in table 2. Both field sites have highly similar lesion sizes in the geometric response variable used for the mixed effects model in, as well as similar lesions sizes across treatments by the different sites. 
</div>
<div class="Standard">
Figure 2 is a boxplot of the geometric response variable across sites and treatments. Variability between trees which were infected appears greater than control trees. Control trees appear to have lower general variation in lesion sizes.
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-3"> </a><div class="figure">
<div class="center">
<img class="embedded" src="BoxplotGeomRespSeperate.png" alt="figure BoxplotGeomRespSeperate.png" style="max-width: 471px; max-height: 376px;">
<div class="caption">
Figure 3 <span class="small"><i>Boxplot of the geometric response as measured in both sites. Points indicate observations outside the inter-quartile range. The effect of treatment is significant (F=161.43, D.F. =1, Pr(&gt;F)= &lt;0.00</i>1)</span>
</div>

</div>

</div>

</div>

</div>
<div class="Standard">
<p><br>
</p>

</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-4.2">4.2</a> Linear Mixed Model for Lesion Geometric Response
</h2>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-4.2.1">4.2.1</a> Model Structure
</h3>
<div class="Standard">
The response variable for the model is the <span class="formula"><i>log</i><sub><i>e</i></sub></span> transformed product of the total length of the lesion and total width of the lesion (see equation 1). Main (fixed) effects for the model structure are the effect of treatment (control or infected), site (Rovaniemi or Lapinjärvi), organ (stem or root), and tissue (phloem or xylem). Additionally, all two way interactions between main effects were included. Random effects included in the model were accounted for in three separate grouping variables; tree level groupings, organ within tree level groupings, and sample within organ within tree level groupings. 
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-4.2.2">4.2.2</a> Fixed Effects
</h3>
<div class="Standard">
Summary of the parameter estimates, confidence intervals for parameter estimates (95%), standard error of the samples parameter estimates, and <i>P</i>-values for the parameter estimates are given in Table 3. 
</div>
<div class="Standard">
The intercept, i.e. the grand mean for all trees is the largest contribution to lesion sizes, and is statistically significant (<i>F</i>=23113.25, <i>P</i>=&lt;0.001). Lesion sizes did not vary significantly between the two field sites (<i>F</i>=0.14, <i>P</i>=0.71), nor did the size of lesions vary between root and stem organs (<i>F</i>=0.47, <i>P</i>=0.49). However, lesions did vary between xylem and phloem tissues (<i>F</i>=353.35, <i>P</i>=&lt;0.001), with xylem tissues having less total necrosis than phloem tissues. No difference in lesion sizes occurred in the interaction between treatment and site. However, the interaction between treatment and organ varied significantly (<i>F</i>=12.95, <i>P</i>=&lt;0.001), with stems which were wounded (mock inoculated) expressing smaller lesions sizes. The interaction between treatment and tissue did not vary significantly (<i>F</i>=0.61,<i> P</i>=0.47). The interaction between site and organ varied marginally, with Rovaniemi stems expressing larger lesion sizes (<i>F</i>=5.99, <i>P</i>=0.02). The response of tissues between sites varied significantly (<i>F</i>=33.52, <i>P</i>=&lt;0.0001), with xylem tissues in Rovaniemi expressing larger lesions. Lastly, the interaction between organs and tissues was significant <i>(F</i>=6.96, <i>P</i>=0.009), indicating that stem xylem tissues expressed smaller lesion sizes. . 
</div>
<div class="Standard">
<span class="environment"></span>
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Table-4"> </a><div class="table">
<div class="caption">
Table 4 Parameter Estimates for Fixed Effects in the Lesion Response Model
</div>
<table>
<tr>
<td align="left" valign="top">
<b><i>Fixed Effect</i></b>
</td>
<td align="center" valign="top">
<b><i>Estimate</i></b>
</td>
<td align="center" valign="top">
<b><i>Standard Error</i></b>
</td>
<td align="center" valign="top">
<b><i>C.I 2.5%*</i></b>
</td>
<td align="center" valign="top">
<b><i> C.I. 97.5%*</i></b>
</td>
<td align="center" valign="top">
<b><i>F</i></b>
</td>
<td align="center" valign="top">
<b><i>P-value**</i></b>
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Intercept</span></i></span></i>
</td>
<td align="center" valign="top">
6.659
</td>
<td align="center" valign="top">
0.096
</td>
<td align="center" valign="top">
6.476 
</td>
<td align="center" valign="top">
6.842
</td>
<td align="center" valign="top">
23113.25
</td>
<td align="center" valign="top">
&lt;0.001
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Treatment-Wounding</span></i></span></i>
</td>
<td align="center" valign="top">
-0.788
</td>
<td align="center" valign="top">
0.127
</td>
<td align="center" valign="top">
-1.039 
</td>
<td align="center" valign="top">
-0.513
</td>
<td align="center" valign="top">
161.44
</td>
<td align="center" valign="top">
&lt;0.001
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Site-Rovaniemi</span></i></span></i>
</td>
<td align="center" valign="top">
-0.270
</td>
<td align="center" valign="top">
0.127
</td>
<td align="center" valign="top">
-0.508 
</td>
<td align="center" valign="top">
-0.020
</td>
<td align="center" valign="top">
0.14
</td>
<td align="center" valign="top">
0.71
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Organ-Stem</span></i></span></i>
</td>
<td align="center" valign="top">
0.084
</td>
<td align="center" valign="top">
0.106
</td>
<td align="center" valign="top">
-0.129
</td>
<td align="center" valign="top">
0.293
</td>
<td align="center" valign="top">
0.47
</td>
<td align="center" valign="top">
0.49
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Tissue-Xylem</span></i></span></i>
</td>
<td align="center" valign="top">
-0.455
</td>
<td align="center" valign="top">
0.043
</td>
<td align="center" valign="top">
-0.534
</td>
<td align="center" valign="top">
 -0.375
</td>
<td align="center" valign="top">
353.35
</td>
<td align="center" valign="top">
&lt;0.001
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Treatment:Wounding*Site:Rovaniemi</span></i></span></i>
</td>
<td align="center" valign="top">
0.055
</td>
<td align="center" valign="top">
0.156
</td>
<td align="center" valign="top">
-0.247
</td>
<td align="center" valign="top">
 0.345
</td>
<td align="center" valign="top">
0.12
</td>
<td align="center" valign="top">
0.72
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Treatment:Wounding*Organ:Stem</span></i></span></i>
</td>
<td align="center" valign="top">
-0.432
</td>
<td align="center" valign="top">
0.120
</td>
<td align="center" valign="top">
-0.656
</td>
<td align="center" valign="top">
-0.187
</td>
<td align="center" valign="top">
12.95
</td>
<td align="center" valign="top">
&lt;0.001
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Treatment:Wounding*Tissue:Xylem</span></i></span></i>
</td>
<td align="center" valign="top">
-0.033
</td>
<td align="center" valign="top">
0.043
</td>
<td align="center" valign="top">
-0.120 
</td>
<td align="center" valign="top">
0.053
</td>
<td align="center" valign="top">
0.61
</td>
<td align="center" valign="top">
0.47
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Site:Rovaniemi*Organ:Stem</span></i></span></i>
</td>
<td align="center" valign="top">
0.294
</td>
<td align="center" valign="top">
0.120
</td>
<td align="center" valign="top">
0.058 
</td>
<td align="center" valign="top">
0.517
</td>
<td align="center" valign="top">
5.99
</td>
<td align="center" valign="top">
0.02
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Site:Rovaniemi*Tissue:Xylem</span></i></span></i>
</td>
<td align="center" valign="top">
0.249
</td>
<td align="center" valign="top">
0.043
</td>
<td align="center" valign="top">
0.164
</td>
<td align="center" valign="top">
0.328
</td>
<td align="center" valign="top">
33.52
</td>
<td align="center" valign="top">
&lt;0.0001
</td>

</tr>
<tr>
<td align="left" valign="top">
<i><span class="small"><i><span class="black">Organ:Stem*Tissue:Xylem</span></i></span></i>
</td>
<td align="center" valign="top">
-0.113
</td>
<td align="center" valign="top">
0.043
</td>
<td align="center" valign="top">
-0.197
</td>
<td align="center" valign="top">
-0.026
</td>
<td align="center" valign="top">
6.96
</td>
<td align="center" valign="top">
0.009
</td>

</tr>

</table>
<i>*C.I. = 95% bootstrapped confidence intervals represented at the 2.5% and 97.5% intervals. Calculated utilizing the &ldquo;confint&rdquo; function in lme4 package (Bates et. al.) with 1000 simulations</i>. <div class="PlainVisible">
<i>** P-value is the adjusted p-value calculated according to the Kenword-Roger approximation to account for multiple comparisons</i>
</div>

</div>

</div>

</div>
<div class="Standard">
<span class="unknown">\end</span>landscape
</div>
<h3 class="Subsubsection">
<a class="toc" name="toc-Subsubsection-4.2.3">4.2.3</a> Random Effects
</h3>
<div class="Standard">
The random effects of the groups and nesting are summarized in Table 4. The standard deviation for nested groups increases with levels of the nesting going from individual trees, to samples nested in organs nested in trees. Residual standard deviation is higher than the standard deviation within trees, and higher than the standard deviation of organ nested within tree. 
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Table-5"> </a><div class="table">
<div class="caption">
Table 5 Random Effects
</div>
<table>
<tr>
<td align="center" valign="top">
<b><i>Random Effects</i></b>
</td>
<td align="center" valign="top">
<b><i>Type</i></b>
</td>
<td align="center" valign="top">
<b><i>Variance</i></b>
</td>
<td align="center" valign="top">
<b><i>Std. Deviation</i></b>
</td>

</tr>
<tr>
<td align="center" valign="top">
Sample:(Organ:Tree)
</td>
<td align="center" valign="top">
Group Intercept
</td>
<td align="center" valign="top">
0.149
</td>
<td align="center" valign="top">
0.386
</td>

</tr>
<tr>
<td align="center" valign="top">
Organ:Tree
</td>
<td align="center" valign="top">
Group Intercept
</td>
<td align="center" valign="top">
0.044
</td>
<td align="center" valign="top">
0.210
</td>

</tr>
<tr>
<td align="center" valign="top">
Tree
</td>
<td align="center" valign="top">
Group Intercept
</td>
<td align="center" valign="top">
0.037
</td>
<td align="center" valign="top">
0.194
</td>

</tr>
<tr>
<td align="center" valign="top">
Residual
</td>
<td align="center" valign="top">
--
</td>
<td align="center" valign="top">
0.083
</td>
<td align="center" valign="top">
0.289
</td>

</tr>

</table>

</div>

</div>

</div>
<div class="Standard">
Plots of the group specific random effects are shown in Figure 3. The plot allows the visualization of the random effects from the model in all the grouping levels. Group labels are not present for the first two grouping dotplots, as there are 360 and 120 distinct group random effects estimated for these, respectively. The highest level of grouping (Tree) has individual labels represented on the y-axis, and the most and least susceptible trees as modeled are easily determined. The variance increases with the levels of grouping; samples within organ have the highest variances, and tree level variance is the lowest of the random effects.
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-4"> </a><div class="figure" style="max-width: 100%;">
<img class="figure" src="ranefplots.png" alt="figure ranefplots.png" style="max-width: 418px; height: 40%; max-height: 452px;">
<div class="caption">
Figure 4 <i>Ordered Dotplots of the Model Random Effects</i>
</div>

</div>

</div>

</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-4.3">4.3</a> Model Validation and Diagnostics
</h2>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-5"> </a><div class="figure">
<img class="embedded" src="DensityPlotParametersPDFBW.png" alt="figure DensityPlotParametersPDFBW.png" style="width: 327px; max-width: 504px; height: 234px; max-height: 360px;">
<div class="caption">
Figure 5 Profiled Likelihood of Parameter Estimates 
</div>

</div>

</div>

</div>
<div class="Standard">
The likelihood of the parameter estimates are presented in Figure 4. Profiles were created utilizing the &ldquo;profile&rdquo; function in lme4 package. Reviewing the ranges and densities of the profile plots gives an indication of the likelihood of the parameter estimate being equal to a value given the observed data, as well as the variability in the likelihood of observing differing values of the parameter estimates. Parameter profiles were normally distributed, with some kurtosis evident in the random effects standard deviation estimates for residual variance, tree, organ within tree, and sample within organ within tree groupings (<span class="formula"><i>σ</i>, <i>σ</i><sub>1</sub>, <i>σ</i><sub>2</sub>, <i>σ</i><sub>3, </sub></span> in Figure 4, respectively). Other profiles of parameter likelihood (e.g. those representing fixed effects) show little to no kurtosis. Ranges of the profiled parameters strongly coincide with the 95% confidence intervals presented in Table 3, and maximal likelihoods in the generated profiles are at the original parameter estimates. 
</div>
<div class="Standard">
Model goodness of fit was examined by plotting the observed values for the geometric response against the fitted values from the model (Figure 5). Marginal <span class="formula"><i>R</i><sup>2</sup></span> is 0.496, or 49.6% of total variance. Conditional \strikeout off\uuline off\uwave off<span class="formula"><i>R</i><sup>2</sup></span> is 0.866, or 86.6% of total variance. The inclusion of the random effects in the model accounts for 0.370, or 37.0% of variance in the data <span class="FootOuter"><span class="SupFootMarker"> [A] </span><span class="HoverFoot"><span class="SupFootMarker"> [A] </span>\strikeout off\uuline off\uwave offThe <span class="default">\uuline default\uwave default<span class="formula"><i>R</i><sup>2</sup></span></span>\strikeout off\uuline off\uwave off measure commonly utilized for linear regressions is not directly applicable to mixed effects models. However, <span class="bibcites">[<a class="bibliocite" name="cite-45" href="#biblio-45"><span class="bib-index">45</span></a>]</span> suggested a new method of calculating <span class="default">\uuline default\uwave default<span class="formula"><i>R</i><sup>2</sup></span></span>\strikeout off\uuline off\uwave off for mixed effects models which separates the influences of the fixed effects and random effects into two measures; marginal <span class="default">\uuline default\uwave default<span class="formula"><i>R</i><sub><i>lmm</i>(<i>m</i>)</sub><sup>2</sup></span> </span>\strikeout off\uuline off\uwave offand conditional <span class="default">\uuline default\uwave default<span class="formula"><i>R</i><sub><i>lmm</i>(<i>c</i>)</sub><sup>2</sup></span></span>\strikeout off\uuline off\uwave off. Marginal <span class="default">\uuline default\uwave default<span class="formula"><i>R</i><sub><i>lmm</i>(<i>m</i>)</sub><sup>2</sup></span> </span>\strikeout off\uuline off\uwave offrepresents the variance attributed to the fixed effects only. Conditional <span class="default">\uuline default\uwave default<span class="formula"><i>R</i><sub><i>lmm</i>(<i>c</i>)</sub><sup>2</sup></span></span>\strikeout off\uuline off\uwave off. includes both the influence from the fixed effects as well as the random effects in the given model. </span></span>.<span class="default">\uuline default\uwave default </span>
</div>
<div class="Standard">
Overall model fit was subjectively good, with 86.6% of total variance explained. The total proportion of variance explained by the fixed effects accounts for a greater proportion of the total variance explained by the model than does the random effects (49.6% versus 37%). However, random effects for the model still account for a large portion of the variance explained in the model, indicating that their inclusion is informative in the context of modeling lesion responses in <i>P. abies</i>.
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-6"> </a><div class="figure">
<div class="center">
<img class="embedded" src="ModelFitPlot.png" alt="figure ModelFitPlot.png" style="width: 209px; max-width: 418px; height: 226px; max-height: 452px;">
<div class="caption">
Figure 6 <i>Plot of the estimated (<span class="formula"><i>ŷ</i></span>) versus observed (<span class="formula"><i>log</i><sub><i>e</i></sub>(<i>y</i>)</span>) values of the geometric response variable from the mixed effects model</i>
</div>

</div>

</div>

</div>

</div>
<div class="Standard">
<p><br>
</p>

</div>
<h1 class="Section">
<a class="toc" name="toc-Section-5">5</a> Discussion
</h1>
<div class="Standard">
Based on the results, this study concludes that differences in lesion sizes are not likely to be due to differences between the sites. However, readers would do well to note that many aspects of this study are subject to analysis only in the context of the collected data and experiment performed herein, and thus broad based inferences upon the effects of historical isolation from pathogen as a potential factor for differences in lesion response in <i>P. abies</i> to <i>H. parviporum</i> should not be drawn from this study alone. These results partially coincide with work carried out by <span class="bibcites">[<a class="bibliocite" name="cite-73" href="#biblio-73"><span class="bib-index">73</span></a>]</span>, which loosely concluded that difference in geographic origin might not affect overall resistance of <i>P. abies </i>to <i>H. annosum s.l.</i> However, their study explored the ability of <i>H. annosum s.s.</i> versus <i>H. parviporum </i>to colonize in different geographical locations with differing temperature regimes, but did not look at regions where the fungus was not known to be present prior to the study. 
</div>
<div class="Standard">
This study found no evidence for the effect of organ on the response in lesion size. However, other studies, including <span class="bibcites">[<a class="bibliocite" name="cite-32" href="#biblio-32"><span class="bib-index">32</span></a>]</span><i>,</i> have concluded that a difference exists in the response of <i>P. abies </i>between roots and stems. Reasons for a lack of similar findings in this study could be attributed to many possible factors, such as soil type, temperature and climate, time between inoculation to harvest, and relative ability of the fungal strain to overcome host responses. However, as the natural pathway for infection of <i>H. annosum s.l.</i> is through the roots, it is somewhat surprising that a difference is not present in this study. One possible explanation for this observation is due to the sizes of roots inoculated. In Rovaniemi, the sizes of roots utilized for the inoculations was significantly smaller than the average root sizes in Lapinjärvi; a study by Garbelotto and Slaughter (1997), found a positive correlation between root diameter and fungal growth. Refitting the model using only data from the Lapinjärvi or Rovaniemi field sites does change the significance of the organ variable; in Lapinjärvi the variable is a significant predictor, whereas in Rovaniemi it is not, indicating that perhaps the smaller sizes of roots in Rovaniemi are the reason for a lack of significance of the organ variable in the model utilizing both field sites. . 
</div>
<div class="Standard">
It is worth noting that most studies where artificial inoculations have been performed on <i>P. abies </i>with <i>H. annosum s.l. </i>focused solely on stem inoculations, which does not mimic the natural infection pathway in <i>H. annosum s.l.</i> Because of this, it can be difficult to judge whether or not a difference is generally assumed to exist in the responses between the two organs based on the results of the few studies which have explored root inoculations. Further complications occur due to the highly variable root structure and variability in root sizes utilized for inoculation points in this study, as well as the lack of control for genotype in this study. 
</div>
<div class="Standard">
The difference in lesion response in tissues was significant in this study. In the study conducted by <span class="bibcites">[<a class="bibliocite" name="cite-32" href="#biblio-32"><span class="bib-index">32</span></a>]</span><i>, </i>no difference was noted between tissues. However, different anatomical features are present in the different tissues, which may influence the ability of <i>H. annosum s.l. </i>to grow in the separate tissues <span class="bibcites">[<a class="bibliocite" name="cite-37" href="#biblio-37"><span class="bib-index">37</span></a>]</span>.  Further reasons for the observed difference in the response between differing tissues could be due to a number of different factors; genetic control of the sample subjects in future experiments could help to determine if the observed differences are due to genetic, environmental, or latent factors. 
</div>
<div class="Standard">
Several of the statistical interactions in this study were found to be significant influences upon the response of <i>P. abies </i>to infection with <i>H. parviporum. </i>The interaction between treatment and organ was significant, with stems being less susceptible to wounding than roots. This is in contrast to other studies which have examined the response of organs to inoculation, which have found significant differences in the treatment by organ, e.g. <span class="bibcites">[<a class="bibliocite" name="cite-32" href="#biblio-32"><span class="bib-index">32</span></a>]</span>, which found stem lesions more susceptible to wounding than root lesions. Furthermore, the interaction between site and organ was found to be marginally significant, with stems lesions in Rovaniemi being larger. This could be due to a number of factors which were unaccounted for in this study. Norway spruce has high genetic plasticity, which could potentially account for differences in the response of the organs at various sites <span class="bibcites">[<a class="bibliocite" name="cite-9" href="#biblio-9"><span class="bib-index">9</span></a>, <a class="bibliocite" name="cite-56" href="#biblio-56"><span class="bib-index">56</span></a>]</span>. However, the design of the study did not control for genotype of phenotype diversity, so there is no way to account for this variance in the study as it was executed. Other possible reasons why a different response by organ was different by site could be due to differences in the overall size of the trees, as trees of differing sizes and ages may respond differently to infections.
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-5.1">5.1</a> Experimental Design Limitations
</h2>
<div class="Standard">
The biggest limitation in the study is the lack of clonal material. Although the results of the study may be sufficient for drawing broad-based conclusions about aspects of the defense response of Norway spruce to infections with <i>H. parviporum</i> in natural settings, it is not possible to conclude what aspects of an individual in the study contributed to more or less susceptibility. Clonal material would have allowed a better assessment of the influence and variability of individual tree’s phenotypes. For example, if a tree on whole shows a strong resistance or susceptibility to the infections, with non-clonal material it is not possible to infer if this is the result of the trees phenotype, or whether it would be expected to see this variation naturally in a sufficiently large sample of clonal material. This lack of clonal material is a principal reason why trees in this study are treated as random effects as opposed to fixed effects; in a study controlled for tree genotype with sufficient numbers of ramets per clones, it would be possible to draw stronger conclusions about the resistance of individual Norway spruce genotypes to <i>H. parviporum</i>.
</div>
<div class="Standard">
The lack of control over genetics makes it difficult to know what to do with outliers present in the data; for this reason, several outliers in the traditional sense were kept in the analysis. Trees RT2 and LT8 showed extreme resistance and susceptibility to treatment, respectively. Tree LW8 had decay due to a natural fungal infection<i> </i>prior to the experiment, but it was only noted when the tree was cut down for processing. If clonal material was utilized, it would be possible to determine if the tree genotype was really susceptible, or if this was an unnatural response to the treatment. Tree RT2 was abnormally resistant to infection, but again it is not possible to deduce the reasons as to why. Because it is not possible to deduce the causes for responses of trees such as RT2 and LW8, which display extreme responses with this studies experimental design, it may not be appropriate to remove the suspected outliers from the analysis; for this reason, these trees and all samples from them were kept in the analysis. 
</div>
<div class="Standard">
The potential effects of historical isolation from the pathogen based on latitude could not accurately be determined based on this study’s experimental design. Too many confounding factors could account for any potential observed differences, even though this study found no difference between tree responses between sites. Additionally, only one site in each geographical region was utilized for the study; to effectively determine the effects of a site factor, more than one replicate for each area (i.e. areas where <i>H. annosum s.l. </i>are known to naturally infect trees, versus one where <i>H. annosum s.l. </i>are not known to be present) is necessary. However, a previous study by <span class="bibcites">[<a class="bibliocite" name="cite-30" href="#biblio-30"><span class="bib-index">30</span></a>]</span> indicated that differences in resistance do exist between geographic areas; two sites in southern Europe utilizing clonal material (Greece and Italy) had significant correlations between fungal growth, lesion size and other indicators of resistance, while a third field site in Sweden utilizing the same clonal material did not share significant correlations with the southern European field sites. However, the authors speculated that environmental factors could be responsible for the lack of correlations between the two regions. Further studies could attempt to address differences between trees acclimatized to certain growing environments by using a crossed design wherein trees from both genetic origins are utilized across field sites.
</div>
<div class="Standard">
Further issues with the experimental design for this study include a lack of control of external factors such as different flora and fauna at the sites could have implications in the estimation of site based effects. This is one major complication with carrying out field based experiments as opposed to those which take place in highly controlled laboratory and greenhouse settings. Temperature, weather events, and other natural influences are not generally controllable in field experiments. However, for drawing ecological conclusions regarding the interactions between Norway spruce and <i>H. annosum s.l., </i>it is useful to carry out field experiments; any resulting residual variances can then be attributed to uncontrollable environmental factors, but would require that the experimental design properly controlled for both the host genotype and site factor. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-5.2">5.2</a> Potential Issues in the Analysis
</h2>
<div class="Standard">
Data analysis, in many instances, becomes a subjective task as opposed to a purely objective one; parsimony is something to strive for in any analysis. In this study, the choices taken in regards to the methodology utilized for analyzing and presenting data were based on balancing several conflicting goals: how to best characterize the lesion response on various levels, while avoiding over parametrization of the model, and maintaining interpretability. 
</div>
<div class="Standard">
In this study, the response of interest is the lesion size as in indication of resistance or susceptibility for a given sample, with inferences then drawn for the different tissues, organs, and individual tree. In reality, the lesion presents as a three dimensional reaction to the inoculation. However, to analyze the lesions, a simplified view of the lesions was utilized, conceptualizing the lesions as a rectangular area based on maximal width and length measurements across the point of inoculation. Lesions from the inoculations presented in many ways, and no single geometric shape is consistent across lesions. In hindsight, there are better, albeit much more time consuming ways, to measure the lesions. With proper photographic equipment and software, it would be possible to measure accurately the entire area of the lesion without needing to resort to constricting the conceptualization of the lesion to a single shape, utilizing a known measurement (i.e., measuring the samples on a white background with a ruler / scale) as a calibration. Implementing this could increase the accuracy of the parameter estimates, as well as better characterize the actual lesions by calculating a highly accurate area measurement. This technique has been applied in other studies with agricultural plants, but to the best of the author’s knowledge, has not been implemented in studies examining lesions created from <i>H. annosum s.l.</i>
</div>
<div class="Standard">
Outliers are of special concern, as their presence in data can modify parameter estimates. In this study, outliers were still included in the analysis. Nevertheless, to ignore the possibility of the influence of several observations which traditionally might be considered outliers would be remiss. The potential influence of outliers is examined in the appendix. 
</div>
<h2 class="Subsection">
<a class="toc" name="toc-Subsection-5.3">5.3</a> Uncertainty in Results
</h2>
<div class="Standard">
Methodologies for performing and assessing the effects of <i>H. annosum s.l</i>. on conifers vary, and thus conclusions and results between studies can be difficult to generalize. Differences in experimental design include ways in which the inoculation is performed. Deep tissue wounding has been used in various studies, and is done by boring directly to the heartwood of a living tree to introduce to the pathogen directly to susceptible tissues, which was the technique utilized for this study <span class="bibcites">[<a class="bibliocite" name="cite-11" href="#biblio-11"><span class="bib-index">11</span></a>]</span>. In a separate inoculation technique, superficial wounding targets only the surface of the cambium of the tree which is inoculated with the pathogen <span class="bibcites">[<a class="bibliocite" name="cite-11" href="#biblio-11"><span class="bib-index">11</span></a>]</span>. Other variables in studies examining the resistance in conifers to <i>H. annosum s.l. </i>include number of replicates per tree, location of field sites, temperature profiles of the regions where experiments are performed, choices of clonal material, length of incubation times, and methods for analyzing the resulting data. However, research indicates that overall sizes of respective lesions in inoculated clones may be a good proxy for indicating resistance or susceptibility to infections from <i>H. annosum s.l.</i> <span class="bibcites">[<a class="bibliocite" name="cite-74" href="#biblio-74"><span class="bib-index">74</span></a>, <a class="bibliocite" name="cite-67" href="#biblio-67"><span class="bib-index">67</span></a>, <a class="bibliocite" name="cite-11" href="#biblio-11"><span class="bib-index">11</span></a>]</span><i>.</i>
</div>
<div class="Standard">
Although the analysis is relatively appropriate for the data collected in this study, and does an acceptable job characterizing the lesions as well as the variance present in the study which are under controllable factors (i.e., treatment, organ, and tissue), the mixed effects approach does little to interpret the potential resistance or susceptibility of the trees on a biological level beyond looking at overall lesion sizes. Furthermore, the lack of control for tree genotype is a troubling shortcoming in this study which prevents anything beyond generalization about tree resistance to be drawn from this data. In future studies, use of clonal material would be essential for comparing the resistance across treatments. Furthermore, molecular methods could be utilized to explore in depth measures of resistance. For example, analysis of RNA transcripts to determine up-regulated gene products such as chitinase, terpenes, and other PR-family proteins, etc., in resistant versus susceptible trees. The use of molecular methods in conjunction with analysis of lesions would provide more detailed information about not only the resistance of trees characterized as a molecular response, but would also allow for correlations to be made between specific gene products and lesion characteristics. Previous studies have incorporated analysis of lesions and fungal growth within host tissues, along with molecular methods for assessing various aspects of tree resistance to <i>H. annosum s.l.</i> <span class="bibcites">[<a class="bibliocite" name="cite-74" href="#biblio-74"><span class="bib-index">74</span></a>, <a class="bibliocite" name="cite-22" href="#biblio-22"><span class="bib-index">22</span></a>]</span>. This study did not analyze the actual growth of the fungus in relation to the size of the lesion response, which could be done in further work as an additional way to characterize the resistance levels of individual trees to infection.
</div>
<div class="Standard">
In hindsight, it is difficult to consider latitude as a reasonable effect to study for the resistance of Norway spruce to <i>H. parviporum</i>; in reality, isolation from historical presence of the pathogen is the parameter this study attempted to address, and subsequent studies should look at this along with specific environmental factors which may influence the pathology during the experiment.
</div>
<h1 class="Section">
<a class="toc" name="toc-Section-6">6</a> Conclusions
</h1>
<div class="Standard">
<i>H. annosum s.l.</i> is a devastating forest pathogen which causes millions of euros in losses to timber products annually. Many factors contribute to the capability of the pathogen to proliferate and spread under natural conditions, and, despite research into forest tree breeding for resistance and use of various control methods, the pathogen still causes huge damages economically, and continues to be a leading concern in the modern forestry sector. Understanding the different aspects of the epidemiology and life cycles of both the host and pathogen may hold key information to combating infections and breeding for improvements in resistance. Factors such as genetic origin, tree age and size, as well as environmental conditions all impact the resistance of Norway spruce to infection to <i>H. annosum s.l.</i>
</div>
<div class="Standard">
The study described in this thesis attempted to address several aspects of the resistance potential of <i>P. abies </i>to <i>H. parviporum </i>in natural conditions. Primary factors examined as the research for this project were the effects of site across areas where the pathogen is both known to be present as well as an area where the pathogen does not have a known historical presence. Additional factors examined in this research include the potential for different responses in both host tree organ (root and stem), and tissues (phloem and xylem), as well as the statistical interactions between various crossed factors included in the design of the experiment. 
</div>
<div class="Standard">
The study found that overall, Norway spruce resistance does not appear to differ between the two field sites, and in this regard, the study concludes that resistance does not appear to vary in Norway spruce populations which are not known to have high rates of natural infection versus area in which the pathogen has a recent historical presence. Future studies should incorporate stricter controls, i.e., the use of clonal materials, and if possible, crossing of geographic origin of the clonal material across environmentally distinct field sites in attempt to better address the influence of geographic isolation from the pathogen in recent history. Further research into site based differences should also account for different environmental and soil based conditions. 
</div>
<div class="Standard">
The study did not find differences with regards to resistance based on the whether the inoculation was performed in the stem versus the root organ of a given tree. This is in contrast to other studies reviewed, which did find a difference between root and stems in their susceptibility. Future studies conducted in a similar matter should incorporate root size measurements as a potential influential variable.
</div>
<div class="Standard">
Findings of a difference in variability of resistance between tissues in this study are in contrast with other studies examining the subject. Exact reasons are uncertain, but possible explanations include genetics, and environmental factors, among others. 
</div>
<div class="Standard">
Finally, the use of a mixed effects model to analyze the complex nature of the data collected proved to be a useful tool to interpret the results, as well as to shed light onto the influences of accounting for random effects in the modeling. Few other studies reviewed have taken this approach to modeling lesion response of forest trees; further studies may attain valuable additional information by including random effects into their analysis and moving away from typical multiple regression / ANOVA based analysis which collected data may violate statistical assumptions for in these sorts of experiments. 
</div>
<div class="Standard">
. 
</div>
<div class="Standard">
<p><br>
</p>

</div>
<div class="Standard">
 <h1 class="biblio">
References
</h1>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-1"><span class="bib-index">1</span></a>] </span> <span class="bib-authors">Sandra L Anagnostakis</span>: “<span class="bib-title">Chestnut Blight: The classical problem of an introduced pathogen</span>”, <i><span class="bib-journal">Mycologia</span></i>, pp. <span class="bib-pages">23—37</span>, <span class="bib-year">1987</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-2"><span class="bib-index">2</span></a>] </span> <span class="bib-authors">F.O. Asiegbu, G. Daniel, M. Johansson</span>: “<span class="bib-title">Defence related reactions of seedling roots of Norway spruce to infection by Heterobasidion annosum (Fr.) Bref.</span>”, <i><span class="bib-journal">Physiological and Molecular Plant Pathology</span></i>, pp. <span class="bib-pages">1—19</span>, <span class="bib-year">1994</span>. URL <a href="http://www.sciencedirect.com/science/article/pii/S0885576505800153"><span class="bib-url">http://www.sciencedirect.com/science/article/pii/S0885576505800153</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-3"><span class="bib-index">3</span></a>] </span> <span class="bib-authors">Fred O. Asiegbu, Aleksandra Adomas, Jan Stenlid</span>: “<span class="bib-title">Conifer root and butt rot caused by Heterobasidion annosum (Fr.) Bref. s.l.</span>”, <i><span class="bib-journal">Molecular Plant Pathology</span></i>, pp. <span class="bib-pages">395—409</span>, <span class="bib-year">2005</span>. URL <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1364-3703.2005.00295.x/full"><span class="bib-url">http://onlinelibrary.wiley.com/doi/10.1111/j.1364-3703.2005.00295.x/full</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-4"><span class="bib-index">4</span></a>] </span> <span class="bib-authors">Baptiste Auguie</span>: <i><span class="bib-title">gridExtra: functions in Grid graphics (R package)</span></i>. <span class="bib-publisher">CRAN</span>, <span class="bib-year">2012</span>. URL <a href="http://cran.r-project.org/web/packages/gridExtra/index.html"><span class="bib-url">http://cran.r-project.org/web/packages/gridExtra/index.html</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-5"><span class="bib-index">5</span></a>] </span> <span class="bib-authors">D Bates, M Maechler, B Bolker</span>: “<span class="bib-title">lme4: Linear mixed-effects models using S4 classes.</span>”, <i><span class="bib-journal">R package version 0.999999-2</span></i>, <span class="bib-year">2013</span>. URL <a href="http://cran.r-project.org/package=lme4"><span class="bib-url">http://cran.r-project.org/package=lme4</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-6"><span class="bib-index">6</span></a>] </span> <span class="bib-authors">Benjamin M. Bolker, Mollie E. Brooks, Connie J. Clark, Shane W. Geange, John R. Poulsen, M. Henry H Stevens, Jada Simone S White</span>: “<span class="bib-title">Generalized linear mixed models: a practical guide for ecology and evolution</span>”, <i><span class="bib-journal">Trends in Ecology and Evolution</span></i>, pp. <span class="bib-pages">127—135</span>, <span class="bib-year">2009</span>. URL <a href="http://www.sciencedirect.com/science/article/pii/S0169534709000196"><span class="bib-url">http://www.sciencedirect.com/science/article/pii/S0169534709000196</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-7"><span class="bib-index">7</span></a>] </span> <span class="bib-authors">Pierluigi Bonello, Thomas R. TR Gordon, Daniel A. Herms, David L. Wood, Nadir Erbilgin</span>: “<span class="bib-title">Nature and ecological implications of pathogen-induced systemic resistance in conifers: a novel hypothesis</span>”, <i><span class="bib-journal">Physiological and Molecular Plant Pathology</span></i>, pp. <span class="bib-pages">95—104</span>, <span class="bib-year">2006</span>. URL <a href="http://linkinghub.elsevier.com/retrieve/pii/S0885576506000877 http://www.sciencedirect.com/science/article/pii/S0885576506000877"><span class="bib-url">http://linkinghub.elsevier.com/retrieve/pii/S0885576506000877 http://www.sciencedirect.com/science/article/pii/S0885576506000877</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-8"><span class="bib-index">8</span></a>] </span> <span class="bib-authors">Thomas E Chase, Robert C Ullrich, C Ullrich</span>: “<span class="bib-title">Five Genes Determining Intersterility in Heterobasidion annosum</span>”, <i><span class="bib-journal">Mycologia</span></i>, pp. <span class="bib-pages">73—81</span>, <span class="bib-year">1990</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-9"><span class="bib-index">9</span></a>] </span> <span class="bib-authors">J Chen, T Källman, X Ma, N Gyllenstrand</span>: “<span class="bib-title">Disentangling the roles of history and local selection in shaping clinal variation of allele frequencies and gene expression in Norway spruce (Picea abies)</span>”, <i><span class="bib-journal">Genetics</span></i>, pp. <span class="bib-pages">865—881</span>, <span class="bib-year">2012</span>. URL <a href="http://www.genetics.org/content/191/3/865.short"><span class="bib-url">http://www.genetics.org/content/191/3/865.short</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-10"><span class="bib-index">10</span></a>] </span> <span class="bib-authors">G. Daniel, F. Asiegbu, M. Johansson</span>: “<span class="bib-title">The saprotrophic wood-degrading abilities of Heterobasidium annosum intersterility groups P and S</span>”, <i><span class="bib-journal">Mycological Research</span></i>, pp. <span class="bib-pages">991—997</span>, <span class="bib-year">1998</span>. URL <a href="http://www.sciencedirect.com/science/article/pii/S095375620860979X"><span class="bib-url">http://www.sciencedirect.com/science/article/pii/S095375620860979X</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-11"><span class="bib-index">11</span></a>] </span> <span class="bib-authors">C Delatour, K von Weissenberg, L Dimitri</span>: <i><span class="bib-title">Host Resistance</span></i> in <i><span class="bib-booktitle">Heterobasidion annosum: Biology, Ecology, Impact and Control</span></i> (<span class="bib-editor">Woodward, S and Stenlid, J and Karjalainen, R and Huttermann, A</span>, ed.). <span class="bib-publisher">CAB International</span>, <span class="bib-year">1998</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-12"><span class="bib-index">12</span></a>] </span> <span class="bib-authors">Alieta Eyles, Pierluigi Bonello, Rebecca Ganley, Caroline Mohammed</span>: “<span class="bib-title">Induced resistance to pests and pathogens in trees</span>”, <i><span class="bib-journal">New Phytologist</span></i>, pp. <span class="bib-pages">893—908</span>, <span class="bib-year">2010</span>. URL <a href="http://dx.doi.org/10.1111/j.1469-8137.2009.03127.x http://www.ncbi.nlm.nih.gov/pubmed/20015067 http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2009.03127.x/full"><span class="bib-url">http://dx.doi.org/10.1111/j.1469-8137.2009.03127.x http://www.ncbi.nlm.nih.gov/pubmed/20015067 http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2009.03127.x/full</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-13"><span class="bib-index">13</span></a>] </span> <span class="bib-authors">Carl Gunnar Fossdal, Nadeem Yaqoob, Paal Krokene, Harald Kvaalen, Halvor Solheim, Igor A Yakovlev</span>: “<span class="bib-title">Local and systemic changes in expression of resistance genes, NB-LRR genes and their putative microRNAs in Norway spruce after wounding and inoculation with the pathogen Ceratocystis polonica.</span>”, <i><span class="bib-journal">BMC plant biology</span></i>, pp. <span class="bib-pages">105</span>, <span class="bib-year">2012</span>. URL <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3431983&tool=pmcentrez&rendertype=abstract"><span class="bib-url">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3431983&tool=pmcentrez&rendertype=abstract</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-14"><span class="bib-index">14</span></a>] </span> <span class="bib-authors">Vincent R. Franceschi, Trygve Krekling, Alan A. Berryman, Erik Christiansen</span>: “<span class="bib-title">Specialized phloem parenchyma cells in Norway spruce (Pinaceae) bark are an important site of defense reactions</span>”, <i><span class="bib-journal">American Journal of Botany</span></i>, pp. <span class="bib-pages">601—615</span>, <span class="bib-year">1998</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-15"><span class="bib-index">15</span></a>] </span> <span class="bib-authors">M. Garbelotto, G. Slaughter</span>: “<span class="bib-title">Secondary spread of Heterobasidion annosum in white fir root-disease centers</span>”, <i><span class="bib-journal">Canadian Journal of Forest Research</span></i>, pp. <span class="bib-pages">766—773</span>, <span class="bib-year">1997</span>. URL <a href="http://www.nrc.ca/cgi-bin/cisti/journals/rp/rp2_abst_e?cjfr_x96-193_27_ns_nf_cjfr27-97 http://www.nrcresearchpress.com/doi/pdf/10.1139/x96-193"><span class="bib-url">http://www.nrc.ca/cgi-bin/cisti/journals/rp/rp2_abst_e?cjfr_x96-193_27_ns_nf_cjfr27-97 http://www.nrcresearchpress.com/doi/pdf/10.1139/x96-193</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-16"><span class="bib-index">16</span></a>] </span> <span class="bib-authors">Matteo Garbelotto, Thomas D Bruns, Fields W Cobb, William J Otrosina</span>: “<span class="bib-title">Differentiation of intersterility groups and geographic provenances among isolates of Heterobasidion annosum detected by random amplified polymorphic DNA assays</span>”, <i><span class="bib-journal">Canadian Journal of Botany</span></i>, pp. <span class="bib-pages">565—569</span>, <span class="bib-year">1993</span>. URL <a href="http://dx.doi.org/10.1139/b93-063"><span class="bib-url">http://dx.doi.org/10.1139/b93-063</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-17"><span class="bib-index">17</span></a>] </span> <span class="bib-authors">Matteo Garbelotto, Paolo Gonthier, Giovanni Nicolotti</span>: “<span class="bib-title">Ecological constraints limit the fitness of fungal hybrids in the Heterobasidion annosum species complex</span>”, <i><span class="bib-journal">Applied and Environmental Microbiology</span></i>, pp. <span class="bib-pages">6106—6111</span>, <span class="bib-year">2007</span>. URL <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2075018&tool=pmcentrez&rendertype=abstract"><span class="bib-url">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2075018&tool=pmcentrez&rendertype=abstract</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-18"><span class="bib-index">18</span></a>] </span> <span class="bib-authors">Matteo Garbelotto, Paolo Gonthier</span>: “<span class="bib-title">Biology, epidemiology, and control of Heterobasidion species worldwide.</span>”, <i><span class="bib-journal">Annual Review of Phytopathology</span></i>, pp. <span class="bib-pages">39—59</span>, <span class="bib-year">2013</span>. URL <a href="http://www.ncbi.nlm.nih.gov/pubmed/23642002"><span class="bib-url">http://www.ncbi.nlm.nih.gov/pubmed/23642002</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-19"><span class="bib-index">19</span></a>] </span> <span class="bib-authors">Gregory S Gilbert</span>: “<span class="bib-title">Evolutionary ecology of plant diseases in natural ecosystems.</span>”, <i><span class="bib-journal">Annual Review of Phytopathology</span></i>, pp. <span class="bib-pages">13—43</span>, <span class="bib-year">2002</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-20"><span class="bib-index">20</span></a>] </span> <span class="bib-authors">Jane Glazebrook</span>: “<span class="bib-title">Contrasting mechanisms of defense against biotrophic and necrotrophic pathogens.</span>”, <i><span class="bib-journal">Annual review of phytopathology</span></i>, pp. <span class="bib-pages">205—27</span>, <span class="bib-year">2005</span>. URL <a href="http://www.annualreviews.org/doi/pdf/10.1146/annurev.phyto.43.040204.135923"><span class="bib-url">http://www.annualreviews.org/doi/pdf/10.1146/annurev.phyto.43.040204.135923</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-21"><span class="bib-index">21</span></a>] </span> <span class="bib-authors">Wickham Hadley</span>: <i><span class="bib-title">ggplot2: elegant graphics for data analysis</span></i>. <span class="bib-publisher">Springer</span>, <span class="bib-year">2009</span>. URL <a href="http://had.co.nz/ggplot2/book"><span class="bib-url">http://had.co.nz/ggplot2/book</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-22"><span class="bib-index">22</span></a>] </span> <span class="bib-authors">Ari M Hietala, Morten Eikenes, Harald Kvaalen, Halvor Solheim, Carl G Fossdal</span>: “<span class="bib-title">Multiplex real-time PCR for monitoring</span>”, <i><span class="bib-journal">Applied and Environmental Microbiology</span></i>, pp. <span class="bib-pages">4413—4420</span>, <span class="bib-year">2003</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-23"><span class="bib-index">23</span></a>] </span> <span class="bib-authors">Ari M Hietala, Harald Kvaalen, Axel Schmidt, Nina Jø hnk, Halvor Solheim, Carl G Fossdal</span>: “<span class="bib-title">Temporal and spatial profiles of chitinase expression by norway spruce in response to bark colonization by Heterobasidion annosum.</span>”, <i><span class="bib-journal">Applied and environmental microbiology</span></i>, pp. <span class="bib-pages">3948—53</span>, <span class="bib-year">2004</span>. URL <a href="http://aem.asm.org/cgi/content/long/70/7/3948"><span class="bib-url">http://aem.asm.org/cgi/content/long/70/7/3948</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-24"><span class="bib-index">24</span></a>] </span> <span class="bib-authors">J. W. Hudgins, E. Christiansen, V. R. Franceschi</span>: “<span class="bib-title">Induction of anatomically based defense responses in stems of diverse conifers by methyl jasmonate: a phylogenetic perspective</span>”, <i><span class="bib-journal">Tree Physiology</span></i>, pp. <span class="bib-pages">251—264</span>, <span class="bib-year">2004</span>. URL <a href="http://treephys.oxfordjournals.org/cgi/content/long/24/3/251"><span class="bib-url">http://treephys.oxfordjournals.org/cgi/content/long/24/3/251</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-25"><span class="bib-index">25</span></a>] </span> <span class="bib-authors">J. W. Hudgins, E. Christiansen, V. R. Franceschi</span>: “<span class="bib-title">Methyl jasmonate induces changes mimicking anatomical defenses in diverse members of the Pinaceae</span>”, <i><span class="bib-journal">Tree Physiology</span></i>, pp. <span class="bib-pages">361—371</span>, <span class="bib-year">2003</span>. URL <a href="http://treephys.oxfordjournals.org/cgi/doi/10.1093/treephys/23.6.361"><span class="bib-url">http://treephys.oxfordjournals.org/cgi/doi/10.1093/treephys/23.6.361</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-26"><span class="bib-index">26</span></a>] </span> <span class="bib-authors">J. W. Hudgins, Steven G. Ralph, Vincent R. Franceschi, Jörg Bohlmann</span>: “<span class="bib-title">Ethylene in induced conifer defense: cDNA cloning, protein expression, and cellular and subcellular localization of 1-aminocyclopropane-1-carboxylate oxidase in resin duct and phenolic parenchyma cells</span>”, <i><span class="bib-journal">Planta</span></i>, pp. <span class="bib-pages">865—877</span>, <span class="bib-year">2006</span>. URL <a href="http://www.ncbi.nlm.nih.gov/pubmed/16705404 http://link.springer.com/article/10.1007/s00425-006-0274-4"><span class="bib-url">http://www.ncbi.nlm.nih.gov/pubmed/16705404 http://link.springer.com/article/10.1007/s00425-006-0274-4</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-27"><span class="bib-index">27</span></a>] </span> <span class="bib-authors">JW Hudgins, VR Franceschi</span>: “<span class="bib-title">Methyl jasmonate-induced ethylene production is responsible for conifer phloem defense responses and reprogramming of stem cambial zone for traumatic resin duct</span>”, <i><span class="bib-journal">Plant Physiology</span></i>, pp. <span class="bib-pages">2134—2149</span>, <span class="bib-year">2004</span>. URL <a href="http://www.plantphysiol.org/content/135/4/2134.short"><span class="bib-url">http://www.plantphysiol.org/content/135/4/2134.short</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-28"><span class="bib-index">28</span></a>] </span> <span class="bib-authors">M. Johansson, J. Stenlid</span>: “<span class="bib-title">Infection of roots of Norway spruce (Picea abies) by Heterobasidion annosum.</span>”, <i><span class="bib-journal">Forest Pathology</span></i>, pp. <span class="bib-pages">32—45</span>, <span class="bib-year">1985</span>. URL <a href="http://doi.wiley.com/10.1111/j.1439-0329.1985.tb01040.x"><span class="bib-url">http://doi.wiley.com/10.1111/j.1439-0329.1985.tb01040.x</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-29"><span class="bib-index">29</span></a>] </span> <span class="bib-authors">Bengt Gunnar Jonsson, Per-Anders Esseen</span>: “<span class="bib-title">Treefall disturbance maintains high bryophyte diversity in a boreal spruce forest</span>”, <i><span class="bib-journal">Journal of Ecology</span></i>, pp. <span class="bib-pages">924—936</span>, <span class="bib-year">1990</span>. URL <a href="http://www.jstor.org/stable/2260943"><span class="bib-url">http://www.jstor.org/stable/2260943</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-30"><span class="bib-index">30</span></a>] </span> <span class="bib-authors">B. Karlsson, P. Tsopelas, L. Zamponi, P. Capretti, N. Soulioti, G. Swedjemark</span>: “<span class="bib-title">Susceptibility to Heterobasidion parviporum in Picea abies clones grown in different environments</span>”, <i><span class="bib-journal">Forest Pathology</span></i>, pp. <span class="bib-pages">83—89</span>, <span class="bib-year">2008</span>. URL <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.2008.00543.x/full"><span class="bib-url">http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.2008.00543.x/full</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-31"><span class="bib-index">31</span></a>] </span> <span class="bib-authors">Bo Karlsson, Gunilla Swedjemark</span>: “<span class="bib-title">Genotypic variation in natural infection frequency of Heterobasidion spp. in a Picea abies clone trial in southern Sweden</span>”, <i><span class="bib-journal">Scandinavian Journal of Forest Research</span></i>, pp. <span class="bib-pages">108—114</span>, <span class="bib-year">2006</span>. URL <a href="http://www.tandfonline.com/doi/abs/10.1080/02827580500529969"><span class="bib-url">http://www.tandfonline.com/doi/abs/10.1080/02827580500529969</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-32"><span class="bib-index">32</span></a>] </span> <span class="bib-authors">S. Keriö, S.M. Niemi, M. Haapanen, G. Daniel, F.O. Asiegbu</span>: “<span class="bib-title">Infection of Picea abies clones with a homokaryotic isolate of Heterobasidion parviporum under field conditions</span>”, <i><span class="bib-journal">Canadian Journal of Forest Research</span></i>, pp. <span class="bib-pages">227—235</span>, <span class="bib-year">2015</span>. URL <a href="http://www.nrcresearchpress.com/doi/10.1139/cjfr-2014-0247"><span class="bib-url">http://www.nrcresearchpress.com/doi/10.1139/cjfr-2014-0247</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-33"><span class="bib-index">33</span></a>] </span> <span class="bib-authors">K Korhonen, P Capretti, R Karjalainen, J Stenlid</span>: <i><span class="bib-title">Distribution of Heterobasidion annosum Intersterility Groups in Europe</span></i> in <i><span class="bib-booktitle">Heterobasidion annosum: Biology, Ecology, Impact and Control</span></i> (<span class="bib-editor">Woodward, S and Stenlid, J and Karjalainen, R and Huttermann, A</span>, ed.). <span class="bib-publisher">CAB International</span>, <span class="bib-year">1998</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-34"><span class="bib-index">34</span></a>] </span> <span class="bib-authors">K. Korhonen, J. Stenlid</span>: <i><span class="bib-title">Biology of Heterobasidion annosum</span></i> in <i><span class="bib-booktitle">Heterobasidion annosum: Biology, ecology, impact and control</span></i> (<span class="bib-editor">Woodward, S and Stenlid, J and Karjalainen, R and Huttermann, A</span>, ed.). <span class="bib-publisher">CAB International</span>, <span class="bib-year">1998</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-35"><span class="bib-index">35</span></a>] </span> <span class="bib-authors">Josef Kostler</span>: <i><span class="bib-title">Silviculture</span></i>. <span class="bib-publisher">Oliver and Boyd</span>, <span class="bib-year">1956</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-36"><span class="bib-index">36</span></a>] </span> <span class="bib-authors">Andriy Kovalchuk, Susanna Keriö, Abbot O Oghenekaro, Emad Jaber, Tommaso Raffaello, Fred O Asiegbu</span>: “<span class="bib-title">Antimicrobial defenses and resistance in forest trees: challenges and perspectives in a genomic era.</span>”, <i><span class="bib-journal">Annual Review of Phytopathology</span></i>, pp. <span class="bib-pages">221—44</span>, <span class="bib-year">2013</span>. URL <a href="http://www.ncbi.nlm.nih.gov/pubmed/23682916"><span class="bib-url">http://www.ncbi.nlm.nih.gov/pubmed/23682916</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-37"><span class="bib-index">37</span></a>] </span> <span class="bib-authors">Trygve Krekling, Vincent R. Franceschi, Paal Krokene, Halvor Solheim</span>: “<span class="bib-title">Differential anatomical response of Norway spruce stem tissues to sterile and fungus infected inoculations</span>”, <i><span class="bib-journal">Trees - Structure and Function</span></i>, pp. <span class="bib-pages">1—9</span>, <span class="bib-year">2004</span>. URL <a href="http://link.springer.com/10.1007/s00468-003-0266-y"><span class="bib-url">http://link.springer.com/10.1007/s00468-003-0266-y</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-38"><span class="bib-index">38</span></a>] </span> <span class="bib-authors">E Lewinsohn, M Gijzen, R Croteau</span>: “<span class="bib-title">Defense mechanisms of conifers : differences in constitutive and wound-induced monoterpene biosynthesis among species.</span>”, <i><span class="bib-journal">Plant Physiology</span></i>, pp. <span class="bib-pages">44—49</span>, <span class="bib-year">1991</span>. URL <a href="http://www.plantphysiol.org/content/96/1/44.short"><span class="bib-url">http://www.plantphysiol.org/content/96/1/44.short</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-39"><span class="bib-index">39</span></a>] </span> <span class="bib-authors">Må rten Lind, Thomas Källman, Jun Chen, Xiao Fei Ma, Jean Bousquet, Michele Morgante, Giusi Zaina, Bo Karlsson, Malin Elfstrand, Martin Lascoux, Jan Stenlid</span>: “<span class="bib-title">A Picea abies linkage map based on SNP markers identifies QTLS for four aspects of resistance to Heterobasidion parviporum infection</span>”, <i><span class="bib-journal">PLoS ONE</span></i>, pp. <span class="bib-pages">e101049</span>, <span class="bib-year">2014</span>. URL <a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0101049#pone.0101049-Arnerup1"><span class="bib-url">http://www.plosone.org/article/info:doi/10.1371/journal.pone.0101049#pone.0101049-Arnerup1</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-40"><span class="bib-index">40</span></a>] </span> <span class="bib-authors">By M Lindberg, M Johansson</span>: “<span class="bib-title">Growth of Heterobasidion annosum through bark of Picea abies</span>”, <i><span class="bib-journal">European Journal of Forest Pathology</span></i>, pp. <span class="bib-pages">377—388</span>, <span class="bib-year">1991</span>. URL <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.1991.tb00775.x/abstract"><span class="bib-url">http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.1991.tb00775.x/abstract</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-41"><span class="bib-index">41</span></a>] </span> <span class="bib-authors">M Lindberg, M Johansson</span>: “<span class="bib-title">Resistance of Picea abies seedlings to infection by Heterobasidion annosum in relation to drought stress</span>”, <i><span class="bib-journal">European Journal of Forest Pathology</span></i>, pp. <span class="bib-pages">115—124</span>, <span class="bib-year">1992</span>. URL <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.1992.tb01438.x/abstract"><span class="bib-url">http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.1992.tb01438.x/abstract</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-42"><span class="bib-index">42</span></a>] </span> <span class="bib-authors">Diane Martin, Dorothea Tholl, Jonathan Gershenzon, Jörg Bohlmann</span>: “<span class="bib-title">Methyl jasmonate induces traumatic resin ducts, terpenoid resin biosynthesis, and terpenoid accumulation in developing xylem of Norway spruce stems.</span>”, <i><span class="bib-journal">Plant Physiology</span></i>, pp. <span class="bib-pages">1003—1018</span>, <span class="bib-year">2002</span>. URL <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=166496&tool=pmcentrez&rendertype=abstract http://www.plantphysiol.org/content/129/3/1003.short"><span class="bib-url">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=166496&tool=pmcentrez&rendertype=abstract http://www.plantphysiol.org/content/129/3/1003.short</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-43"><span class="bib-index">43</span></a>] </span> <span class="bib-authors">Harri Mäkinen, Anna-Maija Hallaksela, Antti Isomäki</span>: “<span class="bib-title">Increment and decay in Norway spruce and Scots pine after artificial logging damage</span>”, <i><span class="bib-journal">Canadian Journal of Forest Research</span></i>, pp. <span class="bib-pages">2130—2141</span>, <span class="bib-year">2007</span>. URL <a href="http://www.nrcresearchpress.com/doi/abs/10.1139/x07-087#.VHRsbckgQQk"><span class="bib-url">http://www.nrcresearchpress.com/doi/abs/10.1139/x07-087#.VHRsbckgQQk</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-44"><span class="bib-index">44</span></a>] </span> <span class="bib-authors">NE Nagy, CG Fossdal, Paal Krokene, Trygve Krekling, Anders Lönnesborg, Halvor Solheim</span>: “<span class="bib-title">Induced responses to pathogen infection in Norway spruce phloem: changes in polyphenolic parenchyma cells, chalcone synthase transcript levels and peroxidase</span>”, <i><span class="bib-journal">Tree Physiology</span></i>, pp. <span class="bib-pages">505—515</span>, <span class="bib-year">2004</span>. URL <a href="http://treephys.oxfordjournals.org/content/24/5/505.short"><span class="bib-url">http://treephys.oxfordjournals.org/content/24/5/505.short</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-45"><span class="bib-index">45</span></a>] </span> <span class="bib-authors">Shinichi Nakagawa, Holger Schielzeth</span>: “<span class="bib-title">A general and simple method for obtaining R̂2 from generalized linear mixed-effects models</span>”, <i><span class="bib-journal">Methods in Ecology and Evolution</span></i>, pp. <span class="bib-pages">133—142</span>, <span class="bib-year">2013</span>. URL <a href="http://doi.wiley.com/10.1111/j.2041-210x.2012.00261.x"><span class="bib-url">http://doi.wiley.com/10.1111/j.2041-210x.2012.00261.x</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-46"><span class="bib-index">46</span></a>] </span> <span class="bib-authors">Anna Napierała-Filipiak, Maciej Filipiak</span>: “<span class="bib-title">Higher resistance of the offspring of Scots pine trees resulting from natural regeneration in old foci of Heterobasidion annosum root rot</span>”, <i><span class="bib-journal">Scandinavian Journal of Forest Research</span></i>, pp. <span class="bib-pages">794—799</span>, <span class="bib-year">2012</span>. URL <a href="http://apps.webofknowledge.com/full_record.do?product=UA&search_mode=CitingArticles&qid=5&SID=1B8mHMHP2H755EOCj@h&page=1&doc=2"><span class="bib-url">http://apps.webofknowledge.com/full_record.do?product=UA&search_mode=CitingArticles&qid=5&SID=1B8mHMHP2H755EOCj@h&page=1&doc=2</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-47"><span class="bib-index">47</span></a>] </span> <span class="bib-authors">G Nicolotti, P Gonthier, G.C. Varese</span>: “<span class="bib-title">Effectiveness of some biocontrol and chemical treatments against Heterobasidion annosum on Norway spruce stumps</span>”, <i><span class="bib-journal">European Journal of Forest Pathology</span></i>, pp. <span class="bib-pages">339—346</span>, <span class="bib-year">1999</span>. URL <a href="http://onlinelibrary.wiley.com/doi/10.1046/j.1439-0329.1999.00159.x/full"><span class="bib-url">http://onlinelibrary.wiley.com/doi/10.1046/j.1439-0329.1999.00159.x/full</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-48"><span class="bib-index">48</span></a>] </span> <span class="bib-authors">Jonàs Oliva, Magnus Thor, Jan Stenlid</span>: “<span class="bib-title">Long-term effects of mechanized stump treatment against Heterobasidion annosum root rot in Picea abies</span>”, <i><span class="bib-journal">Canadian Journal of Forest Research</span></i>, pp. <span class="bib-pages">1020—1033</span>, <span class="bib-year">2010</span>. URL <a href="http://www.nrcresearchpress.com/doi/abs/10.1139/X10-051"><span class="bib-url">http://www.nrcresearchpress.com/doi/abs/10.1139/X10-051</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-49"><span class="bib-index">49</span></a>] </span> <span class="bib-authors">R. B. Pearce</span>: “<span class="bib-title">Antimicrobial defences in the wood of living trees</span>”, <i><span class="bib-journal">New Phytologist</span></i>, pp. <span class="bib-pages">203—233</span>, <span class="bib-year">1996</span>. URL <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.1996.tb01842.x/abstract http://doi.wiley.com/10.1111/j.1469-8137.1996.tb01842.x"><span class="bib-url">http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.1996.tb01842.x/abstract http://doi.wiley.com/10.1111/j.1469-8137.1996.tb01842.x</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-50"><span class="bib-index">50</span></a>] </span> <span class="bib-authors">RB Pearce</span>: “<span class="bib-title">Decay development and its restriction in trees</span>”, <i><span class="bib-journal">Journal of Arboriculture</span></i>, pp. <span class="bib-pages">1—11</span>, <span class="bib-year">2000</span>. URL <a href="http://archive.treelink.org/joa/2000/jan/01_DECAY_DEVELOPMENT_RESTRICTION_IN_TREES_pearce.pdf"><span class="bib-url">http://archive.treelink.org/joa/2000/jan/01_DECAY_DEVELOPMENT_RESTRICTION_IN_TREES_pearce.pdf</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-51"><span class="bib-index">51</span></a>] </span> <span class="bib-authors">T. Piri</span>: <i><span class="bib-title">Inoculum of Heterobasidion parviporum on stump harvested sites</span></i> in <i><span class="bib-booktitle">XIII Conference Root and Butt Rot of Forest Trees IUFRO Working Party 7.02.01</span></i> (<span class="bib-editor">Capretti, P. and Comparini, C. and Garbelotto, M.</span>, ed.). <span class="bib-year">2013</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-52"><span class="bib-index">52</span></a>] </span> <span class="bib-authors">Tuula Piri</span>: <i><span class="bib-title">Silvicultural control of Heterobasidion root rot in Norway spruce forests in southern Finland: Regeneration and vitality fertilization of infected stands</span></i>. <span class="bib-year">2003</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-53"><span class="bib-index">53</span></a>] </span> <span class="bib-authors">Tuula Piri</span>: “<span class="bib-title">The spreading of the S type of Heterobasidion annosum from Norway spruce stumps to the subsequent tree stand</span>”, <i><span class="bib-journal">European Journal of Forest Pathology</span></i>, pp. <span class="bib-pages">193—204</span>, <span class="bib-year">1996</span>. URL <a href="http://doi.wiley.com/10.1111/j.1439-0329.1996.tb00839.x http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.1996.tb00839.x/full"><span class="bib-url">http://doi.wiley.com/10.1111/j.1439-0329.1996.tb00839.x http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0329.1996.tb00839.x/full</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-54"><span class="bib-index">54</span></a>] </span> <span class="bib-authors">D.B. Redfern, J. Stenlid</span>: <i><span class="bib-title">Spore Dispersal and Infection</span></i> in <i><span class="bib-booktitle">Heterobasidion annosum: Biology, Ecology, Impact and Control</span></i> (<span class="bib-editor">Woodward, S. and Stenlid, J. and Karajalainen, R and Huttermann, A</span>, ed.). <span class="bib-publisher">CAB International</span>, <span class="bib-year">1998</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-55"><span class="bib-index">55</span></a>] </span> <span class="bib-authors">D.B. Redfern</span>: “<span class="bib-title">The effect of wood moisture on infection of Sitka spruce stumps by basidiospores of Heterobasidion annosum</span>”, <i><span class="bib-journal">European Journal of Forest Pathology</span></i>, pp. <span class="bib-pages">218—235</span>, <span class="bib-year">1993</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-56"><span class="bib-index">56</span></a>] </span> <span class="bib-authors">PB Reich, Jacek Oleksyn, Jerzy Modrzynsky, Mark Tjoelker</span>: “<span class="bib-title">Evidence that longer needle retention of spruce and pine populations at high elevations and high latitudes is largely a phenotypic response</span>”, <i><span class="bib-journal">Tree Physiology</span></i>, pp. <span class="bib-pages">643—647</span>, <span class="bib-year">1996</span>. URL <a href="http://treephys.oxfordjournals.org/content/16/7/643.short"><span class="bib-url">http://treephys.oxfordjournals.org/content/16/7/643.short</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-57"><span class="bib-index">57</span></a>] </span> <span class="bib-authors">Deepayan Sarkar</span>: <i><span class="bib-title">Lattice: Multivariate Data Visualization with R</span></i>. <span class="bib-publisher">Springer</span>, <span class="bib-year">2008</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-58"><span class="bib-index">58</span></a>] </span> <span class="bib-authors">Nicole M. Scherer, Claudia E. Thompson, Loreta B. Freitas, Sandro L. Bonatto, Francisco M. Salzano</span>: “<span class="bib-title">Patterns of molecular evolution in pathogenesis-related proteins</span>”, <i><span class="bib-journal">Genetics and Molecular Biology</span></i>, pp. <span class="bib-pages">645—653</span>, <span class="bib-year">2005</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-59"><span class="bib-index">59</span></a>] </span> <span class="bib-authors">F. W.M.R. Schwarze</span>: “<span class="bib-title">Development and Prognosis of Decay in the Sapwood of Living Trees</span>”, <i><span class="bib-journal">Arboricultural Journal</span></i>, pp. <span class="bib-pages">321—337</span>, <span class="bib-year">2001</span>. URL <a href="http://www.tandfonline.com/doi/abs/10.1080/03071375.2001.9756454"><span class="bib-url">http://www.tandfonline.com/doi/abs/10.1080/03071375.2001.9756454</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-60"><span class="bib-index">60</span></a>] </span> <span class="bib-authors">Heikki Seppä, Teija Alenius, R. H W Bradshaw, Thomas Giesecke, Maija Heikkilä, Petteri Muukkonen</span>: “<span class="bib-title">Invasion of Norway spruce (Picea abies) and the rise of the boreal ecosystem in Fennoscandia</span>”, <i><span class="bib-journal">Journal of Ecology</span></i>, pp. <span class="bib-pages">629—640</span>, <span class="bib-year">2009</span>. URL <a href="http://doi.wiley.com/10.1111/j.1365-2745.2009.01505.x http://dx.doi.org/10.1111/j.1365-2745.2009.01505.x"><span class="bib-url">http://doi.wiley.com/10.1111/j.1365-2745.2009.01505.x http://dx.doi.org/10.1111/j.1365-2745.2009.01505.x</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-61"><span class="bib-index">61</span></a>] </span> <span class="bib-authors">AL Shigo, HG Marx, DM Carroll</span>: <i><span class="bib-title">Compartmentalization of decay in trees</span></i>. <span class="bib-year">1977</span>. URL <a href="http://arbtalk.co.uk/forum/attachments/picture-forum/33088d1270417726-beauty-c-o-d-i-t-codit-20-20shigo-1-.pdf"><span class="bib-url">http://arbtalk.co.uk/forum/attachments/picture-forum/33088d1270417726-beauty-c-o-d-i-t-codit-20-20shigo-1-.pdf</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-62"><span class="bib-index">62</span></a>] </span> <span class="bib-authors">AL Shigo</span>: “<span class="bib-title">Compartmentalization: a conceptual framework for understanding how trees grow and defend themselves</span>”, <i><span class="bib-journal">Annual review of phytopathology</span></i>, pp. <span class="bib-pages">189—214</span>, <span class="bib-year">1984</span>. URL <a href="http://www.annualreviews.org/doi/pdf/10.1146/annurev.py.22.090184.001201"><span class="bib-url">http://www.annualreviews.org/doi/pdf/10.1146/annurev.py.22.090184.001201</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-63"><span class="bib-index">63</span></a>] </span> <span class="bib-authors">Richard Sniezko, Jason Smith, Jun-Jun Liu, Richard Hamelin</span>: “<span class="bib-title">Genetic resistance to Fusiform rust in Southern pines and White pine blister rust in White pines—a contrasting tale of two rust pathosystems—current status and future prospects</span>”, <i><span class="bib-journal">Forests</span></i>, pp. <span class="bib-pages">2050—2083</span>, <span class="bib-year">2014</span>. URL <a href="http://www.mdpi.com/1999-4907/5/9/2050/"><span class="bib-url">http://www.mdpi.com/1999-4907/5/9/2050/</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-64"><span class="bib-index">64</span></a>] </span> <span class="bib-authors">G Swedjemark, J Stenlid, B Karlsson</span>: “<span class="bib-title">Genetic variation among clones of Picea abies in resistance to growth of Heterobasidion annosum</span>”, <i><span class="bib-journal">Silvae Genetica</span></i>, pp. <span class="bib-pages">206—212</span>, <span class="bib-year">1998</span>. URL <a href="http://sauerlaender-verlag.com/fileadmin/content/dokument/archiv/silvaegenetica/46_1997/46-6-369.pdf"><span class="bib-url">http://sauerlaender-verlag.com/fileadmin/content/dokument/archiv/silvaegenetica/46_1997/46-6-369.pdf</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-65"><span class="bib-index">65</span></a>] </span> <span class="bib-authors">Gunilla Swedjemark, Bo Karlsson, Jan Stenlid</span>: “<span class="bib-title">Exclusion of Heterobasidion parviporum from inoculated clones of Picea abies and evidence of systemic induced resistance</span>”, <i><span class="bib-journal">Scandinavian Journal of Forest Research</span></i>, pp. <span class="bib-pages">110—117</span>, <span class="bib-year">2007</span>. URL <a href="http://www.tandfonline.com/doi/abs/10.1080/02827580701262774"><span class="bib-url">http://www.tandfonline.com/doi/abs/10.1080/02827580701262774</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-66"><span class="bib-index">66</span></a>] </span> <span class="bib-authors">Gunilla Swedjemark, Bo Karlsson</span>: “<span class="bib-title">Genotypic variation in susceptibility following artificial Heterobasidion annosum inoculation of Picea abies clones in a 17-year-old field test</span>”, <i><span class="bib-journal">Scandinavian Journal of Forest Research</span></i>, pp. <span class="bib-pages">103—111</span>, <span class="bib-year">2004</span>. URL <a href="http://www.informaworld.com/openurl?genre=article&doi=10.1080/02827580310018032&magic=crossref||D404A21C5BB053405B1A640AFFD44AE3"><span class="bib-url">http://www.informaworld.com/openurl?genre=article&doi=10.1080/02827580310018032&magic=crossref||D404A21C5BB053405B1A640AFFD44AE3</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-67"><span class="bib-index">67</span></a>] </span> <span class="bib-authors">G Swedjmark, J Stenlid</span>: “<span class="bib-title">Between-tree and between-isolate variation for growth of S-group Heterobasidion annosum in sapwood of Picea abies cuttings</span>”, <i><span class="bib-journal">Canadian Journal of Forest Research</span></i>, pp. <span class="bib-pages">711—715</span>, <span class="bib-year">1997</span>. URL <a href="http://www.nrcresearchpress.com/doi/abs/10.1139/x96-191"><span class="bib-url">http://www.nrcresearchpress.com/doi/abs/10.1139/x96-191</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-68"><span class="bib-index">68</span></a>] </span> <span class="bib-authors">Magnus Thor, Jan Stenlid</span>: “<span class="bib-title">Heterobasidion annosum infection of Picea abies following manual or mechanized stump treatment</span>”, <i><span class="bib-journal">Scandinavian Journal of Forest Research</span></i>, pp. <span class="bib-pages">154—164</span>, <span class="bib-year">2005</span>. URL <a href="http://dx.doi.org/10.1080/02827580510008338"><span class="bib-url">http://dx.doi.org/10.1080/02827580510008338</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-69"><span class="bib-index">69</span></a>] </span> <span class="bib-authors">Mari Mette Tollefsrud, Roy Kissling, Felix Gugerli, Ø ystein Johnsen, Tore Skrø ppa, Rachid Cheddadi, W. O. Van der Knaap, Małgorzata Latałowa, Ruth Terhürne-Berson, Thomas Litt, Thomas Geburek, Christian Brochmann, Christoph Sperisen</span>: “<span class="bib-title">Genetic consequences of glacial survival and postglacial colonization in Norway spruce: Combined analysis of mitochondrial DNA and fossil pollen</span>”, <i><span class="bib-journal">Molecular Ecology</span></i>, pp. <span class="bib-pages">4134—4150</span>, <span class="bib-year">2008</span>. URL <a href="http://doi.wiley.com/10.1111/j.1365-294X.2008.03893.x"><span class="bib-url">http://doi.wiley.com/10.1111/j.1365-294X.2008.03893.x</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-70"><span class="bib-index">70</span></a>] </span> <span class="bib-authors">Hd VanEtten, Jw Mansfield, Ja Bailey, Ee Farmer</span>: “<span class="bib-title">Two classes of plant antibiotics: Phytoalexins versus "phytoanticipins"</span>”, <i><span class="bib-journal">The Plant Cell</span></i>, pp. <span class="bib-pages">1191—1192</span>, <span class="bib-year">1994</span>. URL <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=160512&tool=pmcentrez&rendertype=abstract"><span class="bib-url">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=160512&tool=pmcentrez&rendertype=abstract</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-71"><span class="bib-index">71</span></a>] </span> <span class="bib-authors">Rimvydas Vasaitis, Jan Stenlid, IM Thomsen</span>: “<span class="bib-title">Stump removal to control root rot in forest stands: a literature study</span>”, <i><span class="bib-journal">Silva Fennica</span></i>, pp. <span class="bib-pages">457—483</span>, <span class="bib-year">2008</span>. URL <a href="http://www.metla.fi/silvafennica/full/sf42/sf423457.pdf"><span class="bib-url">http://www.metla.fi/silvafennica/full/sf42/sf423457.pdf</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-72"><span class="bib-index">72</span></a>] </span> <span class="bib-authors">Radha Veluthakkal, M.G.` Modhumita Ghosh Dasgupta</span>: “<span class="bib-title">Pathogenesis-related genes and proteins in forest tree species</span>”, <i><span class="bib-journal">Trees</span></i>, pp. <span class="bib-pages">993—1006</span>, <span class="bib-year">2010</span>. URL <a href="http://link.springer.com/10.1007/s00468-010-0489-7"><span class="bib-url">http://link.springer.com/10.1007/s00468-010-0489-7</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-73"><span class="bib-index">73</span></a>] </span> <span class="bib-authors">Johanna Witzell, Mattias Berglund, Jonas Rönnberg</span>: “<span class="bib-title">Does temperature regime govern the establishment of Heterobasidion annosum in Scandinavia?</span>”, <i><span class="bib-journal">International Journal of Biometeorology</span></i>, pp. <span class="bib-pages">275—84</span>, <span class="bib-year">2011</span>. URL <a href="http://www.ncbi.nlm.nih.gov/pubmed/20524013"><span class="bib-url">http://www.ncbi.nlm.nih.gov/pubmed/20524013</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-74"><span class="bib-index">74</span></a>] </span> <span class="bib-authors">S. Woodward, S. Bianchi, W. J. A. Bodles, L. Beckett, M. Michelozzi</span>: “<span class="bib-title">Physical and chemical responses of Sitka spruce (Picea sitchensis) clones to colonization by Heterobasidion annosum as potential markers for relative host susceptibility</span>”, <i><span class="bib-journal">Tree Physiology</span></i>, pp. <span class="bib-pages">1701—1710</span>, <span class="bib-year">2007</span>. URL <a href="http://treephys.oxfordjournals.org/cgi/content/long/27/12/1701"><span class="bib-url">http://treephys.oxfordjournals.org/cgi/content/long/27/12/1701</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-75"><span class="bib-index">75</span></a>] </span> <span class="bib-authors">S. Woodward, J. Stenlid, R. Karjalainen, A. Hüttermann</span>: <i><span class="bib-title">Heterobasidion annosum: Biology, Ecology, Impact and Control.</span></i> (<span class="bib-editor">Woodward, S and Stenlid, J and Karjalainen, R and Huttermann, A</span>, ed.). <span class="bib-publisher">CAB International</span>, <span class="bib-year">1998</span>. URL <a href="http://www.cabdirect.org/abstracts/19981007711.html;jsessionid=7F408758F27D39F66A1AA8AD27878D57?freeview=true#"><span class="bib-url">http://www.cabdirect.org/abstracts/19981007711.html;jsessionid=7F408758F27D39F66A1AA8AD27878D57?freeview=true#</span></a>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-76"><span class="bib-index">76</span></a>] </span> <span class="bib-authors">Steve Woodward, Jan Stenlid, K Korhonen, A Huttermann</span>: <i><span class="bib-title">Preface</span></i> in <i><span class="bib-booktitle">Heterobasidion annosum: Biology, Ecology, Impact and Control</span></i> (<span class="bib-editor">Woodward, S and Stenlid, J and Karjalainen, R and Huttermann, A</span>, ed.). <span class="bib-year">1998</span>.
</p>
<p class="biblio">
<span class="entry">[<a class="biblioentry" name="biblio-77"><span class="bib-index">77</span></a>] </span> <span class="bib-authors">Esa Ylitalo</span>: <i><span class="bib-title">Forest Resources</span></i> in <i><span class="bib-booktitle">Finnish Statistical Yearbook of Forestry</span></i>. <span class="bib-publisher">Finnish Forest Research Institute</span>, <span class="bib-year">2013</span>.
</p>

</div>
<div class="Standard">
<p><br>
</p>

</div>
<h1 class="Section-">
<a class="toc" name="toc-Appendix--1"></a>Appendix 1
</h1>
<div class="Standard">
  
</div>
<h2 class="Subsection-">
<a class="toc" name="toc-Subsection--1"></a>Maps
</h2>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-1"> </a><div class="figure">
<img class="embedded" src="TopoLapinjarvi.png" alt="figure TopoLapinjarvi.png" style="width: 549px; max-width: 785px; height: 354px; max-height: 507px;">
<div class="caption">
Figure 1 Map of Lapinjärvi Field Site
</div>

</div>

</div>

</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-2"> </a><div class="figure">
<img class="embedded" src="TopoRovaniemi.png" alt="figure TopoRovaniemi.png" style="width: 549px; max-width: 785px; height: 354px; max-height: 507px;">
<div class="caption">
Figure 2 Map of Rovaniemi Field Site
</div>

</div>

</div>
<a class="IndexReference" name="entry-Map-of-Lapinjärvi-Field-Site-Map-of-Rovaniemi-Field-Site-0" href="#index-Map-of-Lapinjärvi-Field-Site-Map-of-Rovaniemi-Field-Site">↓</a>
</div>
<div class="Standard">
<p><br>
</p>

</div>
<h1 class="Section-">
<a class="toc" name="toc-Appendix--2"></a>Appendix 2 
</h1>
<h2 class="Subsection-">
<a class="toc" name="toc-Subsection--2"></a>Normality of Data and Outliers
</h2>
<div class="Standard">
The geometric response variable violated assumptions of normality. A natural logarithm transform was applied to the data resulting in a much closer to normal structure of the data.
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-3"> </a><div class="figure">
<img class="embedded" src="GeometricResponseHistograms.png" alt="figure GeometricResponseHistograms.png" style="width: 292px; max-width: 418px; height: 316px; max-height: 452px;">
<div class="caption">
Figure 3 <i>Histogram of the geometric response variable, <span class="formula"><i>y</i></span>, and transformed geometric response variable <span class="formula"><i>log</i><sub><i>e</i></sub>(<i>y</i>)</span>, indicating a lack of normality in the uncorrected data.</i>
</div>

</div>

</div>

</div>
<div class="Standard">
To determine if any trees had significant influence on the parameter estimates, post hoc investigation of the random effect of tree level groupings was performed. Cook’s distance is a measure which is utilized to examine the influence of a unit on all parameters. Using the value of <span class="formula">4 ⁄ <i>n</i></span>, where n is the number of groups (in this case, trees) gives a cutoff value of <span class="formula">0.066</span> for determining overly influential observations. Only tree RT2 is overly influential (see Figure A.4) when considering the influence upon all fixed effects. 
</div>
<h1 class="Section-">
<a class="toc" name="toc-Appendix--3"></a>Appendix 2
</h1>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-4"> </a><div class="figure">
<img class="embedded" src="CooksDist_Tree.png" alt="figure CooksDist_Tree.png" style="width: 232px; max-width: 358px; height: 343px; max-height: 529px;">
<div class="caption">
Figure 4 Cook’s Distance for all Trees 
</div>

</div>

</div>

</div>
<div class="Standard">
<p><br>
</p>

</div>
<h1 class="Section-">
<a class="toc" name="toc-Appendix--4"></a>Appendix 3
</h1>
<h2 class="Subsection-">
<a class="toc" name="toc-Subsection--3"></a>Model Comparison
</h2>
<div class="Standard">
Several models were constructed before settling upon a final model. To compare models, an ANOVA was used to select evaluate the most informative model. However, as the calculation of degrees of freedom with restricted estimate of maximum likelihood for random effects is non-trivial, so models were refit using maximum likelihood. Three models were compared; the most elementary model included only tree level random effects groupings, and is specified in equation (3). The next model included full specification of random effects as in the full model, but did not include any interaction terms between fixed effects (4). The final model contained full specifications of the random effects as well as all two way interactions between fixed effects (5). The full model including interactions as well as full specification of random effects is significantly better than the other models (<span class="formula">χ<sup>2</sup></span>= 271.13, <i>P = &lt; 0.001).</i>
</div>
<div class="Standard">
<span class="footnotesize"><div class="formula">
<a class="eqnumber" name="eq:Minimal Model">(3) </a><i>ŷ</i> = <i>Site</i> + <i>Treatment</i> + <i>Organ</i> + <i>Sample</i> + <i>random</i>(<i>Tree</i>)
</div>
<div class="formula">
<a class="eqnumber" name="eq:No Interactions">(4) </a><i>ŷ</i> = <i>Site</i> + <i>Treatment</i> + <i>Organ</i> + <i>Tissue</i> + <i>random</i>(<i>Tree</i> ⁄ <i>Organ</i> ⁄ <i>Sample</i>)
</div>
<div class="formula">
<a class="eqnumber" name="eq:Full Model">(5) </a><span class="environment"><span class="arrayrow">
<span class="arraycell align-l">
<i>ŷ</i> = <i>Site</i>*<i>Treatment</i> + <i>Site</i>*<i>Organ</i> + <i>Site</i>*<i>Tissue</i> + <i>Treatment</i>*<i>Organ</i> + 
</span>

</span>
<span class="arrayrow">
<span class="arraycell align-l">
 
</span>

</span>
<span class="arrayrow">
<span class="arraycell align-l">
<i>Treatment</i>*<i>Tissue</i> + <i>Organ</i>*<i>Tissue</i> + <i>random</i>(<i>Tree</i> ⁄ <i>Organ</i> ⁄ <i>Sample</i>)
</span>

</span>
</span>
</div>
</span>
</div>
<div class="Standard">
<div class="float">
<a class="Label" name="Figure-5"> </a><div class="figure">
<div class="caption">
Figure 5 Model Comparisons Table
</div>
<table>
<tr>
<td align="left" valign="top">
Model Description
</td>
<td align="left" valign="top">
Pr &gt; <span class="formula">χ<sup>2</sup></span>
</td>

</tr>
<tr>
<td align="left" valign="top">
<span class="small">(3) No Interactions, single random effect (Tree)</span>
</td>
<td align="left" valign="top">
<span class="small">--</span>
</td>

</tr>
<tr>
<td align="left" valign="top">
<span class="small">(4) No Interactions, full random effects</span>
</td>
<td align="left" valign="top">
<span class="small">1</span>
</td>

</tr>
<tr>
<td align="left" valign="top">
<span class="small">(5) Full model utilized in the study</span>
</td>
<td align="left" valign="top">
<span class="small">&lt; 0.001</span>
</td>

</tr>

</table>

</div>

</div>

</div>

<hr class="footer">
<div class="footer" id="generated-by">
Document generated by <a href="http://elyxer.nongnu.org/">eLyXer 1.2.5 (2013-03-10)</a> on <span class="create-date">2015-09-06T15:18:39.847000</span>
</div>
</div>
</body>
</html>
